<!DOCTYPE html>
<!-- saved from url=(0068)https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><link rel="stylesheet" type="text/css" href="./pubmed_files/jig.min.css">
    
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="./pubmed_files/output.7e0f021e7705.css" type="text/css">
  
  <link rel="stylesheet" href="./pubmed_files/output.b5beb09202a8.css" type="text/css"><link rel="stylesheet" href="./pubmed_files/output.58b52be466ef.css" type="text/css"><link rel="stylesheet" href="./pubmed_files/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="./pubmed_files/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="./pubmed_files/font-awesome.min.css">
  
<link type="text/css" href="./pubmed_files/base.6dc1ceca1f0e73d17eca.css" rel="stylesheet">


    <link rel="stylesheet" href="./pubmed_files/stixfonts.css" type="text/css"><link rel="stylesheet" href="./pubmed_files/pmcrefs1.min.css" type="text/css"><link rel="stylesheet" href="./pubmed_files/ncbi_web.min.css" type="text/css"><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="./pubmed_files/article.634482294d06e2dd698c.css" rel="stylesheet">


    <title>Histone H3 trimethylation at lysine 36 guides m6A RNA modification co-transcriptionally - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico">
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png">

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png">

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend">

<meta name="ncbi_db" content="pmc">

<meta name="ncbi_phid" content="939B8C751BDA84D500003CE450D41771.1.m_2">


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article">


    

    

    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Histone H3 trimethylation at lysine 36 guides m6A RNA modification co-transcriptionally"><meta name="citation_authors" content="Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao, Mingli Sun, Zhenhua Chen, Xiaolan Deng, Gang Xiao, Franziska Auer, Lars Klemm, Huizhe Wu, Zhixiang Zuo, Xi Qin, Yunzhu Dong, Yile Zhou, Hanjun Qin, Shu Tao, Juan Du, Jun Liu, Zhike Lu, Hang Yin, Ana Mesquita, Celvie L. Yuan, Yueh-Chiang Hu, Wenju Sun, Rui Su, Lei Dong, Chao Shen, Chenying Li, Ying Qing, Xi Jiang, Xiwei Wu, Miao Sun, Jun-Lin Guan, Lianghu Qu, Minjie Wei, Markus Müschen, Gang Huang, Chuan He, Jianhua Yang, Jianjun Chen"><meta name="citation_date" content="March 2019"><meta name="citation_issue" content="7748"><meta name="citation_volume" content="567"><meta name="citation_firstpage" content="414"><meta name="citation_doi" content="10.1038/s41586-019-1016-7"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6438714/?report=abstract"><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC6438714/"><meta name="citation_pmid" content="30867593"><meta name="DC.Title" content="Histone H3 trimethylation at lysine 36 guides m6A RNA modification co-transcriptionally"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Huilin Huang"><meta name="DC.Contributor" content="Hengyou Weng"><meta name="DC.Contributor" content="Keren Zhou"><meta name="DC.Contributor" content="Tong Wu"><meta name="DC.Contributor" content="Boxuan Simen Zhao"><meta name="DC.Contributor" content="Mingli Sun"><meta name="DC.Contributor" content="Zhenhua Chen"><meta name="DC.Contributor" content="Xiaolan Deng"><meta name="DC.Contributor" content="Gang Xiao"><meta name="DC.Contributor" content="Franziska Auer"><meta name="DC.Contributor" content="Lars Klemm"><meta name="DC.Contributor" content="Huizhe Wu"><meta name="DC.Contributor" content="Zhixiang Zuo"><meta name="DC.Contributor" content="Xi Qin"><meta name="DC.Contributor" content="Yunzhu Dong"><meta name="DC.Contributor" content="Yile Zhou"><meta name="DC.Contributor" content="Hanjun Qin"><meta name="DC.Contributor" content="Shu Tao"><meta name="DC.Contributor" content="Juan Du"><meta name="DC.Contributor" content="Jun Liu"><meta name="DC.Contributor" content="Zhike Lu"><meta name="DC.Contributor" content="Hang Yin"><meta name="DC.Contributor" content="Ana Mesquita"><meta name="DC.Contributor" content="Celvie L. Yuan"><meta name="DC.Contributor" content="Yueh-Chiang Hu"><meta name="DC.Contributor" content="Wenju Sun"><meta name="DC.Contributor" content="Rui Su"><meta name="DC.Contributor" content="Lei Dong"><meta name="DC.Contributor" content="Chao Shen"><meta name="DC.Contributor" content="Chenying Li"><meta name="DC.Contributor" content="Ying Qing"><meta name="DC.Contributor" content="Xi Jiang"><meta name="DC.Contributor" content="Xiwei Wu"><meta name="DC.Contributor" content="Miao Sun"><meta name="DC.Contributor" content="Jun-Lin Guan"><meta name="DC.Contributor" content="Lianghu Qu"><meta name="DC.Contributor" content="Minjie Wei"><meta name="DC.Contributor" content="Markus Müschen"><meta name="DC.Contributor" content="Gang Huang"><meta name="DC.Contributor" content="Chuan He"><meta name="DC.Contributor" content="Jianhua Yang"><meta name="DC.Contributor" content="Jianjun Chen"><meta name="DC.Date" content="2019 Mar"><meta name="DC.Identifier" content="10.1038/s41586-019-1016-7"><meta name="DC.Language" content="en"><meta property="og:title" content="Histone H3 trimethylation at lysine 36 guides m6A RNA modification co-transcriptionally"><meta property="og:type" content="article"><meta property="og:description" content="DNA and histone modifications exhibit noticeable impacts on gene expression[] . Being the most prevalent internal modification in mRNA, N[6] -Methyladenosine (m[6] A) mRNA modification emerges as an important post-transcriptional mechanism of gene regulation[-] ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png?_=0"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data">


<meta name="ncbi_nwds_ver" content="1.1.9-2"><meta name="ncbi_nwds" content="yes"><script id="pingerInjectedGTM"></script><script async="1" src="./pubmed_files/analytics.js"></script><style>.ncbi-alerts .ncbi-alert__shutdown-outer { position: relative; background-color: #f4e3db; border-left: 8px solid #d54309;  background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg xmlns='http://www.w3.org/2000/svg' width='126' height='126' viewBox='0 0 126 126'%3E%3Cpath fill='%231B1B1B' d='M117.18,31.592 C111.585,22.006 103.995,14.416 94.409,8.821 C84.821,3.226 74.354,0.429 63.001,0.429 C51.649,0.429 41.18,3.226 31.593,8.821 C22.006,14.415 14.416,22.005 8.821,31.592 C3.225,41.179 0.428,51.649 0.428,63 C0.428,74.351 3.226,84.82 8.82,94.408 C14.415,103.992 22.005,111.584 31.592,117.179 C41.179,122.774 51.648,125.571 63,125.571 C74.352,125.571 84.822,122.774 94.408,117.179 C103.994,111.585 111.584,103.994 117.179,94.408 C122.773,84.82 125.57,74.35 125.57,63 C125.57,51.649 122.773,41.178 117.18,31.592 Z M73.43,102.025 C73.43,102.786 73.184,103.423 72.696,103.939 C72.208,104.455 71.61,104.712 70.903,104.712 L55.26,104.712 C54.554,104.712 53.929,104.441 53.386,103.898 C52.843,103.355 52.572,102.73 52.572,102.025 L52.572,86.546 C52.572,85.84 52.843,85.215 53.386,84.672 C53.929,84.129 54.554,83.858 55.26,83.858 L70.903,83.858 C71.61,83.858 72.209,84.116 72.696,84.631 C73.184,85.149 73.43,85.785 73.43,86.546 L73.43,102.025 Z M73.266,73.999 C73.211,74.542 72.927,75.018 72.412,75.425 C71.895,75.832 71.258,76.035 70.498,76.035 L55.425,76.035 C54.664,76.035 54.012,75.832 53.469,75.425 C52.926,75.018 52.654,74.542 52.654,73.999 L51.269,23.404 C51.269,22.751 51.54,22.263 52.083,21.937 C52.627,21.503 53.279,21.285 54.039,21.285 L71.965,21.285 C72.726,21.285 73.377,21.502 73.92,21.937 C74.463,22.263 74.733,22.752 74.733,23.404 L73.266,73.999 Z'/%3E%3C/svg%3E%0A");}.ncbi-alerts .ncbi-alert__info-outer { position: relative; background-color: #e7f6f8; border-left: 8px solid #00bde3; background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg xmlns='http://www.w3.org/2000/svg' width='126' height='126' viewBox='0 0 126 126'%3E%3Cpath fill='%231B1B1B' d='M117.18,31.592 C111.585,22.006 103.995,14.416 94.409,8.821 C84.821,3.226 74.354,0.429 63.001,0.429 C51.649,0.429 41.179,3.226 31.593,8.821 C22.006,14.415 14.416,22.005 8.821,31.592 C3.225,41.179 0.428,51.649 0.428,63 C0.428,74.352 3.226,84.82 8.82,94.408 C14.415,103.993 22.005,111.584 31.592,117.179 C41.179,122.774 51.648,125.571 63,125.571 C74.352,125.571 84.822,122.774 94.408,117.179 C103.994,111.585 111.584,103.994 117.179,94.408 C122.773,84.821 125.57,74.351 125.57,63 C125.57,51.648 122.773,41.178 117.18,31.592 Z M52.572,16.071 C52.572,15.31 52.816,14.686 53.305,14.197 C53.794,13.709 54.419,13.464 55.179,13.464 L70.823,13.464 C71.583,13.464 72.208,13.709 72.695,14.197 C73.183,14.686 73.429,15.31 73.429,16.071 L73.429,29.107 C73.429,29.867 73.183,30.492 72.695,30.98 C72.208,31.469 71.583,31.713 70.823,31.713 L55.179,31.713 C54.419,31.713 53.794,31.469 53.305,30.98 C52.816,30.492 52.572,29.867 52.572,29.107 L52.572,16.071 Z M83.857,102.107 C83.857,102.867 83.611,103.492 83.124,103.979 C82.637,104.468 82.012,104.712 81.25,104.712 L44.75,104.712 C43.989,104.712 43.365,104.468 42.876,103.979 C42.387,103.491 42.143,102.866 42.143,102.106 L42.143,89.07 C42.143,88.308 42.387,87.685 42.876,87.196 C43.365,86.708 43.99,86.463 44.75,86.463 L52.572,86.463 L52.572,60.392 L44.75,60.392 C43.989,60.392 43.365,60.148 42.876,59.659 C42.387,59.171 42.143,58.546 42.143,57.785 L42.143,44.75 C42.143,43.989 42.387,43.365 42.876,42.876 C43.365,42.387 43.99,42.143 44.75,42.143 L70.823,42.143 C71.583,42.143 72.208,42.387 72.695,42.876 C73.183,43.365 73.429,43.989 73.429,44.75 L73.429,86.464 L81.249,86.464 C82.01,86.464 82.635,86.709 83.123,87.197 C83.61,87.685 83.856,88.31 83.856,89.071 L83.856,102.107 L83.857,102.107 Z'/%3E%3C/svg%3E");}.ncbi-alerts div[class$="-outer"]{background-position: 28px 20px;  background-size: 32px 32px;  background-repeat: no-repeat;padding:20px 20px 20px 28px;}.ncbi-alerts div[class$="-inner"]{padding-left: 52px; padding-right: 52px;}@media (max-width: 639px){.ncbi-alerts div[class$="outer"]{background-position: 15px 20px;}.ncbi-alerts div[class$="-inner"]{padding-left: 0px; padding-right: 0px; padding-top: 40px;}}.ncbi-alerts button.close{cursor: pointer;position:absolute; top: 10px; right: 20px; width: 36px; height: 32px; border:0; background-color: transparent; background-image: url("data:image/svg+xml,%3Csvg version='1.2' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' overflow='visible' preserveAspectRatio='none' viewBox='0 0 24 24' width='32' height='32'%3E%3Cg%3E%3Cpath xmlns:default='http://www.w3.org/2000/svg' id='window-close' d='M14.9,16.42c-0.13,0.13-0.33,0.14-0.47,0.01c0,0-0.01-0.01-0.01-0.01L12,14l-2.43,2.42 c-0.13,0.13-0.33,0.14-0.47,0.01c0,0-0.01-0.01-0.01-0.01L7.58,14.9c-0.13-0.13-0.14-0.33-0.01-0.47c0,0,0.01-0.01,0.01-0.01L10,12 L7.58,9.57C7.45,9.45,7.44,9.24,7.57,9.1c0,0,0.01-0.01,0.01-0.01L9.1,7.58c0.13-0.13,0.33-0.14,0.47-0.01c0,0,0.01,0.01,0.01,0.01 L12,10l2.43-2.43c0.13-0.13,0.33-0.14,0.47-0.01c0,0,0.01,0.01,0.01,0.01l1.51,1.53c0.13,0.13,0.14,0.33,0.01,0.47 c0,0-0.01,0.01-0.01,0.01L14,12l2.43,2.43c0.13,0.13,0.14,0.33,0.01,0.47c0,0-0.01,0.01-0.01,0.01L14.9,16.42L14.9,16.42z M20.84,4.49C20.53,4.17,20.1,3.99,19.66,4H4.34C3.42,4,2.67,4.75,2.67,5.67l0,0v12.66c0,0.92,0.75,1.67,1.67,1.67l0,0h15.32 c0.92,0,1.67-0.75,1.67-1.67l0,0V5.67C21.34,5.23,21.16,4.8,20.84,4.49z' style='fill: rgb(33, 33, 33);' vector-effect='non-scaling-stroke'/%3E%3C/g%3E%3C/svg%3E");background-size: 32px 32px; background-repeat:no-repeat;background-position:center center}.ncbi-alerts button.close:focus{ outline: 1px dotted #000000;}@media (max-width: 639px){.ncbi-alerts button.close{right: 0px;}}.ncbi-alerts p{margin-bottom: 0px; margin-top: 0px; font-size: 18px; line-height: 28px;}.ncbi-alerts .list-items > p{font-size: 0.94em; display: inline;}.ncbi-alerts .list-items > p a {font-weight: 700;}.ncbi-alerts .list-items > p:not(:last-child)::after{margin:0 8px;content:'|'}@media (min-width: 768px) and (max-width: 991px){.ncbi-alerts .list-items > p:nth-child(2)::after{content:'\a'!important;white-space: pre;}}@media(max-width: 767px){.ncbi-alerts .list-items > p{display:block;}.ncbi-alerts .list-items > p::after{content:''!important;}}#shutdown_2021 {margin-bottom: 10px;}</style></head>
<body id="ui-ncbiexternallink-3"><section class="pmc-alerts">
    
    
</section><section data-section="Alerts">
	<div class="ncbi-alerts-placeholder" style="margin-bottom: 0px;"></div><div class="ncbi-alerts nCoV_shutdown" id="nCoV_shutdown" role="region" aria-label="Alert" style="margin-bottom: 0px; display: none;"><div class="ncbi-alert__shutdown-outer"><div class="ncbi-alert__shutdown-inner"><p><strong>COVID-19 Information</strong></p><div class="list-items"><p><a href="https://cdc.gov/coronavirus">Public health information (CDC)</a></p><p><a href="https://covid19.nih.gov/">Research information (NIH)</a></p><p><a href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">SARS-CoV-2 data (NCBI)</a></p><p><a href="https://combatcovid.hhs.gov/">Prevention and treatment information (HHS)</a></p><p><a href="https://salud.nih.gov/covid-19/">Español</a></p></div></div><button type="button" class="close" aria-controls="nCoV_shutdown" aria-label="Close Alert">&nbsp;</button></div></div>
</section>

   
    
        
    
    
        <button class="back-to-top back-to-top--base" data-ga-category="pagination" data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="./pubmed_files/favicon-57.png" alt="U.S. flag">
        <p>An official website of the United States government</p>
        <button class="usa-accordion-button usa-banner-button" aria-expanded="false" aria-controls="gov-banner-top">
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div class="usa-banner-content usa-grid usa-accordion-content" id="gov-banner-top" aria-hidden="true">
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img class="usa-banner-icon usa-media_block-img" src="./pubmed_files/icon-dot-gov.svg" alt="Dot gov">
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br>
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img class="usa-banner-icon usa-media_block-img" src="./pubmed_files/icon-https.svg" alt="Https">
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br>
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="./pubmed_files/AgencyLogo.svg" alt="NIH NLM Logo">
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F%3Freport%3Dclassic" class="usa-button header-button" style="" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none" aria-controls="account_popup" aria-haspopup="true" aria-expanded="false">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true" tabindex="-1" aria-modal="true" style="display: none;">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="https://www.ncbi.nlm.nih.gov/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="https://www.ncbi.nlm.nih.gov/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="https://www.ncbi.nlm.nih.gov/account/signout/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F%3Freport%3Dclassic" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov/" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="https://www.ncbi.nlm.nih.gov/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>

<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/" title="Home" aria-label="Home"></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1" aria-hidden="true">
            <div role="search" class="pmc-header--stretch">
    <form action="https://www.ncbi.nlm.nih.gov/pmc/" class="usa-search " autocomplete="off">
      <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
      <span class="clearable">
        <span class="twitter-typeahead ncbi-autocomplete"><input required="required" autocomplete-url="/pmc/autocomplete/pmc/" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" class="tt-input" spellcheck="false" dir="auto" style="position: relative; vertical-align: top;" aria-owns="pmc-search_listbox" role="combobox" aria-autocomplete="list" aria-expanded="false"><span role="status" aria-live="polite" style="position: absolute; padding: 0px; border: 0px; height: 1px; width: 1px; margin-bottom: -1px; margin-right: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap;"></span><pre aria-hidden="true" style="position: absolute; visibility: hidden; white-space: pre; font-family: Roboto, &quot;Helvetica Neue&quot;, Arial, Tahoma; font-size: 16px; font-style: normal; font-variant: normal; font-weight: 400; word-spacing: 0px; letter-spacing: 0px; text-indent: 0px; text-rendering: auto; text-transform: none;"></pre><div role="listbox" class="tt-menu" id="pmc-search_listbox" style="position: absolute; top: 100%; left: 0px; z-index: 100; display: none;"><div role="presentation" class="tt-dataset tt-dataset-0"></div></div></span>
          <i class="clear-btn"></i>
      </span>
      <button type="submit">
        <span class="usa-search-submit-text">Search</span>
      </button>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item">
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item">
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item" aria-current="page">
                    PMC6438714
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                


    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/pdf/nihms-1521376.pdf" class="int-view">PDF (6.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li class="citation-link other_item"><a class="citationexporter ctxp" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" data-citationid="PMC6438714" role="button" aria-expanded="false" aria-haspopup="true">Cite</a></li>

            <li>
                
                    




    <div class="favorites-button-container" id="pmc-favorites-button" data-toggle-favorites-enabled="false" data-add-to-favorites-url-hash="#add-to-favorites" data-in-favorites="false" data-toggle-url="https://www.ncbi.nlm.nih.gov/pmc/toggle-favorites-collection/PMC6438714/" data-collections-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/" data-remove-from-favorites-text="Remove from Favorites collection" data-add-to-favorites-text="Add to Favorites collection" data-display-title-attr="Favorites">
        
            <a href="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A//www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/%3Freport%3Dclassic%23add-to-favorites" class="usa-button favorites-button favorites-button-empty favorites-button-link" id="pmc-favorites-button-link" aria-label="Add to Favorites collection">
                Favorites
            </a>
        
    </div>


                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F&amp;text=Histone%20H3%20trimethylation%20at%20lysine%2036%20guides%20m6A%20RNA%20modification%20co-transcriptionally" alt="Share on Twitter" aria-expanded="false" aria-haspopup="true"><i class="fa fa-twitter fa-stack-1x">&nbsp;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F" alt="Share on Facebook" aria-expanded="false" aria-haspopup="true"><i class="fa fa-facebook fa-stack-1x">&nbsp;</i></a></li>
                <li>
                    <div class="share-permalink dropdown-block">
                        <button class="trigger" alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg">
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" aria-hidden="true">
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    
        <section>
            <h6>RESOURCES</h6>
            <ul class="pmc-sidebar__pubmed_links">
                <li><a href="https://pubmed.ncbi.nlm.nih.gov/?linkname=pubmed_pubmed&amp;from_uid=30867593">Similar articles in PubMed</a></li>
            </ul>
        </section>
    

            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item">
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item">
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item" aria-current="page">
                    PMC6438714
                </li>
            
    </ul>
 

        </section>
        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="./pubmed_files/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area target="_blank" rel="noopener noreferrer" shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> 
                
            
        </section>
        <section role="document">
             
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box report-classic"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden,.nomenu&#39;" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span role="menubar"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature.</a></span> Author manuscript; available in PMC 2019 Sep 13.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=30867593" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature. 2019 Mar; 567(7748): 414–419. </a></div><span class="fm-vol-iss-date">Published online 2019 Mar 13. </span> <span class="doi"><span>doi:&nbsp;</span><a href="https://dx.doi.org/10.1038%2Fs41586-019-1016-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41586-019-1016-7</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6438714</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS1521376</span></div><div class="fm-citation-pmid">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30867593">30867593</a></div></div></div><h1 class="content-title">Histone H3 trimethylation at lysine 36 guides m<sup>6</sup>A RNA modification co-transcriptionally</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Huang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452392482144" co-class="co-affbox">Huilin Huang</a>,<sup>1,</sup><sup>2,</sup><sup>13</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Weng%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452394527840" co-class="co-affbox">Hengyou Weng</a>,<sup>1,</sup><sup>2,</sup><sup>13</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhou%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452406769520" co-class="co-affbox">Keren Zhou</a>,<sup>3,</sup><sup>4,</sup><sup>13</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452389572624" co-class="co-affbox">Tong Wu</a>,<sup>5,</sup><sup>13</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhao%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452404714128" co-class="co-affbox">Boxuan Simen Zhao</a>,<sup>5,</sup><sup>13</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452405807456" co-class="co-affbox">Mingli Sun</a>,<sup>1,</sup><sup>6</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Chen%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452390976016" co-class="co-affbox">Zhenhua Chen</a>,<sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Deng%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452390974032" co-class="co-affbox">Xiaolan Deng</a>,<sup>1,</sup><sup>2,</sup><sup>6</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Xiao%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452405143264" co-class="co-affbox">Gang Xiao</a>,<sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Auer%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452391752800" co-class="co-affbox">Franziska Auer</a>,<sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Klemm%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452388555376" co-class="co-affbox">Lars Klemm</a>,<sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452385096464" co-class="co-affbox">Huizhe Wu</a>,<sup>1,</sup><sup>2,</sup><sup>6</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zuo%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452393735440" co-class="co-affbox">Zhixiang Zuo</a>,<sup>2,</sup><sup>7</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qin%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452388195408" co-class="co-affbox">Xi Qin</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Dong%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452403231424" co-class="co-affbox">Yunzhu Dong</a>,<sup>8</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhou%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452393205520" co-class="co-affbox">Yile Zhou</a>,<sup>8</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qin%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452393203536" co-class="co-affbox">Hanjun Qin</a>,<sup>9</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Tao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452393955744" co-class="co-affbox">Shu Tao</a>,<sup>9</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Du%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452392946592" co-class="co-affbox">Juan Du</a>,<sup>9</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452404015824" co-class="co-affbox">Jun Liu</a>,<sup>5</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Lu%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452388659728" co-class="co-affbox">Zhike Lu</a>,<sup>5</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yin%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452386367344" co-class="co-affbox">Hang Yin</a>,<sup>5</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Mesquita%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452394096576" co-class="co-affbox">Ana Mesquita</a>,<sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yuan%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452393902624" co-class="co-affbox">Celvie L. Yuan</a>,<sup>10</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Hu%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452390316992" co-class="co-affbox">Yueh-Chiang Hu</a>,<sup>10</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452390315008" co-class="co-affbox">Wenju Sun</a>,<sup>3,</sup><sup>4</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Su%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452326401136" co-class="co-affbox">Rui Su</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Dong%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452385955472" co-class="co-affbox">Lei Dong</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Shen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452390978208" co-class="co-affbox">Chao Shen</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Li%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452388542192" co-class="co-affbox">Chenying Li</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qing%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452351386800" co-class="co-affbox">Ying Qing</a>,<sup>1,</sup><sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Jiang%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452403594032" co-class="co-affbox">Xi Jiang</a>,<sup>1,</sup><sup>2,</sup><sup>11</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452391206912" co-class="co-affbox">Xiwei Wu</a>,<sup>9</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452403903072" co-class="co-affbox">Miao Sun</a>,<sup>12</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Guan%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452403901088" co-class="co-affbox">Jun-Lin Guan</a>,<sup>2</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452406316624" co-class="co-affbox">Lianghu Qu</a>,<sup>3,</sup><sup>4</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452403798480" co-class="co-affbox">Minjie Wei</a>,<sup>6</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=M%26%23x000fc%3Bschen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452392027792" co-class="co-affbox">Markus Müschen</a>,<sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Huang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452392792592" co-class="co-affbox">Gang Huang</a>,<sup>8,</sup><sup>*</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=He%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452404740896" co-class="co-affbox">Chuan He</a>,<sup>5,</sup><sup>*</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452350844496" co-class="co-affbox">Jianhua Yang</a>,<sup>3,</sup><sup>4,</sup><sup>*</sup> and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Chen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593" class="affpopup" co-rid="_co_idm140452386255760" co-class="co-affbox">Jianjun Chen</a><sup>1,</sup><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140452392482144"><h3 class="no_margin">Huilin Huang</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Huang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Huilin Huang</a></div></div><div id="_co_idm140452394527840"><h3 class="no_margin">Hengyou Weng</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Weng%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Hengyou Weng</a></div></div><div id="_co_idm140452406769520"><h3 class="no_margin">Keren Zhou</h3><p><sup>3</sup>Key Laboratory of Gene Engineering of the Ministry of Education, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><p><sup>4</sup>State Key Laboratory for Biocontrol, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><p><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhou%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Keren Zhou</a></div></div><div id="_co_idm140452389572624"><h3 class="no_margin">Tong Wu</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><p><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Tong Wu</a></div></div><div id="_co_idm140452404714128"><h3 class="no_margin">Boxuan Simen Zhao</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><p><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhao%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Boxuan Simen Zhao</a></div></div><div id="_co_idm140452405807456"><h3 class="no_margin">Mingli Sun</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>6</sup>Department of Pharmacology, School of Pharmacy, China Medical University, Shenyang 110122, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Mingli Sun</a></div></div><div id="_co_idm140452390976016"><h3 class="no_margin">Zhenhua Chen</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Chen%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Zhenhua Chen</a></div></div><div id="_co_idm140452390974032"><h3 class="no_margin">Xiaolan Deng</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>6</sup>Department of Pharmacology, School of Pharmacy, China Medical University, Shenyang 110122, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Deng%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Xiaolan Deng</a></div></div><div id="_co_idm140452405143264"><h3 class="no_margin">Gang Xiao</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Xiao%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Gang Xiao</a></div></div><div id="_co_idm140452391752800"><h3 class="no_margin">Franziska Auer</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Auer%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Franziska Auer</a></div></div><div id="_co_idm140452388555376"><h3 class="no_margin">Lars Klemm</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Klemm%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Lars Klemm</a></div></div><div id="_co_idm140452385096464"><h3 class="no_margin">Huizhe Wu</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>6</sup>Department of Pharmacology, School of Pharmacy, China Medical University, Shenyang 110122, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Huizhe Wu</a></div></div><div id="_co_idm140452393735440"><h3 class="no_margin">Zhixiang Zuo</h3><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>7</sup>Sun Yat-sen University Cancer Center, State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Guangzhou, 510060, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zuo%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Zhixiang Zuo</a></div></div><div id="_co_idm140452388195408"><h3 class="no_margin">Xi Qin</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qin%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Xi Qin</a></div></div><div id="_co_idm140452403231424"><h3 class="no_margin">Yunzhu Dong</h3><p><sup>8</sup>Divisions of Pathology and Experimental Hematology and Cancer Biology, Cincinnati Children's Hospital Medical Center, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Dong%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Yunzhu Dong</a></div></div><div id="_co_idm140452393205520"><h3 class="no_margin">Yile Zhou</h3><p><sup>8</sup>Divisions of Pathology and Experimental Hematology and Cancer Biology, Cincinnati Children's Hospital Medical Center, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhou%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Yile Zhou</a></div></div><div id="_co_idm140452393203536"><h3 class="no_margin">Hanjun Qin</h3><p><sup>9</sup>Intergrative Genomics Core, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qin%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Hanjun Qin</a></div></div><div id="_co_idm140452393955744"><h3 class="no_margin">Shu Tao</h3><p><sup>9</sup>Intergrative Genomics Core, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Tao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Shu Tao</a></div></div><div id="_co_idm140452392946592"><h3 class="no_margin">Juan Du</h3><p><sup>9</sup>Intergrative Genomics Core, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Du%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Juan Du</a></div></div><div id="_co_idm140452404015824"><h3 class="no_margin">Jun Liu</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Jun Liu</a></div></div><div id="_co_idm140452388659728"><h3 class="no_margin">Zhike Lu</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Lu%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Zhike Lu</a></div></div><div id="_co_idm140452386367344"><h3 class="no_margin">Hang Yin</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yin%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Hang Yin</a></div></div><div id="_co_idm140452394096576"><h3 class="no_margin">Ana Mesquita</h3><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Mesquita%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Ana Mesquita</a></div></div><div id="_co_idm140452393902624"><h3 class="no_margin">Celvie L. Yuan</h3><p><sup>10</sup>Division of Developmental Biology, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yuan%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Celvie L. Yuan</a></div></div><div id="_co_idm140452390316992"><h3 class="no_margin">Yueh-Chiang Hu</h3><p><sup>10</sup>Division of Developmental Biology, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Hu%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Yueh-Chiang Hu</a></div></div><div id="_co_idm140452390315008"><h3 class="no_margin">Wenju Sun</h3><p><sup>3</sup>Key Laboratory of Gene Engineering of the Ministry of Education, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><p><sup>4</sup>State Key Laboratory for Biocontrol, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Wenju Sun</a></div></div><div id="_co_idm140452326401136"><h3 class="no_margin">Rui Su</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Su%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Rui Su</a></div></div><div id="_co_idm140452385955472"><h3 class="no_margin">Lei Dong</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Dong%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Lei Dong</a></div></div><div id="_co_idm140452390978208"><h3 class="no_margin">Chao Shen</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Shen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Chao Shen</a></div></div><div id="_co_idm140452388542192"><h3 class="no_margin">Chenying Li</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Li%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Chenying Li</a></div></div><div id="_co_idm140452351386800"><h3 class="no_margin">Ying Qing</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qing%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Ying Qing</a></div></div><div id="_co_idm140452403594032"><h3 class="no_margin">Xi Jiang</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><p><sup>11</sup>Department of Pharmacology, and Bone Marrow Transplantation Center of the First Affiliated Hospital, Zhejiang University School of Medicine; Hangzhou, Zhejiang 310058, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Jiang%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Xi Jiang</a></div></div><div id="_co_idm140452391206912"><h3 class="no_margin">Xiwei Wu</h3><p><sup>9</sup>Intergrative Genomics Core, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Xiwei Wu</a></div></div><div id="_co_idm140452403903072"><h3 class="no_margin">Miao Sun</h3><p><sup>12</sup>Department of Pediatrics, University of Cincinnati College of Medicine; Division of Human Genetics, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sun%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Miao Sun</a></div></div><div id="_co_idm140452403901088"><h3 class="no_margin">Jun-Lin Guan</h3><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Guan%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Jun-Lin Guan</a></div></div><div id="_co_idm140452406316624"><h3 class="no_margin">Lianghu Qu</h3><p><sup>3</sup>Key Laboratory of Gene Engineering of the Ministry of Education, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><p><sup>4</sup>State Key Laboratory for Biocontrol, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Lianghu Qu</a></div></div><div id="_co_idm140452403798480"><h3 class="no_margin">Minjie Wei</h3><p><sup>6</sup>Department of Pharmacology, School of Pharmacy, China Medical University, Shenyang 110122, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Minjie Wei</a></div></div><div id="_co_idm140452392027792"><h3 class="no_margin">Markus Müschen</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=M%26%23x000fc%3Bschen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Markus Müschen</a></div></div><div id="_co_idm140452392792592"><h3 class="no_margin">Gang Huang</h3><p><sup>8</sup>Divisions of Pathology and Experimental Hematology and Cancer Biology, Cincinnati Children's Hospital Medical Center, OH 45229, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Huang%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Gang Huang</a></div></div><div id="_co_idm140452404740896"><h3 class="no_margin">Chuan He</h3><p><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=He%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Chuan He</a></div></div><div id="_co_idm140452350844496"><h3 class="no_margin">Jianhua Yang</h3><p><sup>3</sup>Key Laboratory of Gene Engineering of the Ministry of Education, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><p><sup>4</sup>State Key Laboratory for Biocontrol, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Jianhua Yang</a></div></div><div id="_co_idm140452386255760"><h3 class="no_margin">Jianjun Chen</h3><p><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p><p><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Chen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30867593">Jianjun Chen</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" class="pmctoggle" rid="idm140452347496272_ai">Author information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" class="pmctoggle" rid="idm140452347496272_cpl">Copyright and License information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/" style="margin-left: 1em">Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140452347496272_ai" style="display:none"><div class="fm-affl" lang="en" id="A1"><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</div><div class="fm-affl" lang="en" id="A2"><sup>2</sup>Department of Cancer Biology, University of Cincinnati College of Medicine, Cincinnati, OH 45219, USA</div><div class="fm-affl" lang="en" id="A3"><sup>3</sup>Key Laboratory of Gene Engineering of the Ministry of Education, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</div><div class="fm-affl" lang="en" id="A4"><sup>4</sup>State Key Laboratory for Biocontrol, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</div><div class="fm-affl" lang="en" id="A5"><sup>5</sup>Department of Chemistry, Department of Biochemistry and Molecular Biology, Institute for Biophysical Dynamics, Howard Hughes Medical Institute, University of Chicago, Chicago, IL 60637, USA</div><div class="fm-affl" lang="en" id="A6"><sup>6</sup>Department of Pharmacology, School of Pharmacy, China Medical University, Shenyang 110122, China</div><div class="fm-affl" lang="en" id="A7"><sup>7</sup>Sun Yat-sen University Cancer Center, State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Guangzhou, 510060, China</div><div class="fm-affl" lang="en" id="A8"><sup>8</sup>Divisions of Pathology and Experimental Hematology and Cancer Biology, Cincinnati Children's Hospital Medical Center, OH 45229, USA</div><div class="fm-affl" lang="en" id="A9"><sup>9</sup>Intergrative Genomics Core, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</div><div class="fm-affl" lang="en" id="A10"><sup>10</sup>Division of Developmental Biology, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</div><div class="fm-affl" lang="en" id="A11"><sup>11</sup>Department of Pharmacology, and Bone Marrow Transplantation Center of the First Affiliated Hospital, Zhejiang University School of Medicine; Hangzhou, Zhejiang 310058, China</div><div class="fm-affl" lang="en" id="A12"><sup>12</sup>Department of Pediatrics, University of Cincinnati College of Medicine; Division of Human Genetics, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</div><div class="fm-affl" lang="en" id="A13"><sup>13</sup>These authors contributed equally: Huilin Huang, Hengyou Weng, Keren Zhou, Tong Wu, Boxuan Simen Zhao.</div><div id="CR1"><sup>*</sup>Correspondence should be addressed to J.C., J.Y., C.H., G.H.. (<a href="mailto:dev@null" data-email="gro.hoc@nehcnaij" class="oemail">gro.hoc@nehcnaij</a>; <a href="mailto:dev@null" data-email="nc.ude.usys.liam@7hjgnay" class="oemail">nc.ude.usys.liam@7hjgnay</a>; <a href="mailto:dev@null" data-email="ude.ogacihcu@ehnauhc" class="oemail">ude.ogacihcu@ehnauhc</a>; <a href="mailto:dev@null" data-email="gro.cmhcc@gnauH.gnaG" class="oemail">gro.cmhcc@gnauH.gnaG</a>).</div><div><p id="P1" class="p p-first">Author Contributions:</p>H.H., H.Weng, and J.C. conceived and designed the entire project. H.H., H.Weng, G.H., C.H., J.Y. and J.C. designed and supervised the research. H.H. and H.Weng prepared all samples for HT-seq, T.W. and B.S.Z. performed H3K36me3 ChIP-seq and m<sup>6</sup>A-seq with input from H.Y., T.W. performed PAR-CLIP-seq, miCLIP-seq and ribosome profiling, H.Weng performed METTL14 ChIP-seq, H.H. performed mRNA stability profiling. H.H. and H.Weng performed dot blot and gene specific ChIP and MeRIP assays. H.H. and H.Weng prepared samples for LC-MS/MS while B.S.Z. and T.W. performed mass spectrometry. H.H. performed CLIP assays, gel shift, peptide pull-down, immunofluorescence, in vitro differentiation of mESCs, and gene expression correlation analyses. H.Weng isolated nascent RNA. H.H. and H.Weng performed MST. H.H. and H.Weng performed co-IP with the input from Z.C. H.H. performed western blot with the help of H.Weng, Mingli S., Z.C., X.D., H.Wu and X.Q. H.H. performed qPCR with the help of H.Weng, Mingli S., H.Wu and X.Q. G.X. designed sgRNAs and prepared dCas9 cell samples with the help of L.K. and F.A., while H.H. and H.Weng detected m<sup>6</sup>A and H3K36me3 levels in specific loci. H.Weng constructed plasmids with the input from H.H., Mingli S. and X.Q. H.H. and H.Weng prepared all SETD2 KD and OE cell lines, A.M. established SETD2-KO Hela cells, F.A. established METTL14-KO HepG2 cells. Y.D., Y.Z. and G.H. provided Setd2-KO mice and Setd2 inducible KO mESCs. T.W. purified METTL3 and METTL14 recombinant protein with the help from J.L. Z.Z. performed integrated analysis of ChIP-seq and m<sup>6</sup>A-seq data using public data. K.Z. analyzed all the genome-wide and transcriptome-wide data with input from W.S. and supervision from J.Y. and H.H. H.Q., S.T., J.D., C.Y., Y.H., R.S., L.D., C.S., C.L., Y.Q., X.J., X.W., Miao S., J.G., L.Q., M.W., M.M., G.H., C.H., J.Y., and J.C. contributed reagents/analytic tools and/or grant support; H.H., H.Weng and J.C. wrote the manuscript and all authors commented on the manuscript.</div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140452347496272_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Users may view, print, copy, and download text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use:<a href="http://www.nature.com/authors/editorial_policies/license.html#terms">http://www.nature.com/authors/editorial_policies/license.html#terms</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available  at <a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=30867593" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature</a></div><div>See other articles in PMC that <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/citedby/">cite</a> the published article.</div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-1.docx" data-ga-action="click_feat_suppl">NIHMS1521376-supplement-1.docx</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;39E02003-6556-4AC0-95E0-2E861B03C88F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-2.pdf" data-ga-action="click_feat_suppl">NIHMS1521376-supplement-2.pdf</a><span style="color:gray"> (46M)</span></div><div class="small guid">GUID:&nbsp;9C0DA48D-E118-4AB5-ABFF-1621BAFECCF2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-3.docx" data-ga-action="click_feat_suppl">NIHMS1521376-supplement-3.docx</a><span style="color:gray"> (56K)</span></div><div class="small guid">GUID:&nbsp;1409D08D-7C13-487B-AECA-FE3442C68EE2</div></div></dd></dl></div><div id="ABS1" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ABS1title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">DNA and histone modifications exhibit noticeable impacts on gene expression<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621532" role="button" aria-expanded="false" aria-haspopup="true">1</a></sup>. Being the most prevalent internal modification in mRNA, <em>N</em><sup>6</sup>-Methyladenosine (m<sup>6</sup>A) mRNA modification emerges as an important post-transcriptional mechanism of gene regulation<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621541" role="button" aria-expanded="false" aria-haspopup="true">2</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621531" role="button" aria-expanded="false" aria-haspopup="true">4</a></sup> and plays critical roles in various normal and pathological bioprocesses<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621530" role="button" aria-expanded="false" aria-haspopup="true">5</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621544" role="button" aria-expanded="false" aria-haspopup="true">12</a></sup>. However, how m<sup>6</sup>A is precisely and dynamically deposited in the transcriptome remains elusive. Here we report that H3K36me3 histone modification, a marker for transcription elongation, globally guides m<sup>6</sup>A modification. We found that m<sup>6</sup>A modifications enrich in the vicinity of H3K36me3 peaks, and are reduced globally when cellular H3K36me3 is depleted. Mechanistically, H3K36me3 is recognized and bound directly by METTL14, a critical component of the m<sup>6</sup>A methyltransferase complex (MTC), which in turn facilitates the binding of the m<sup>6</sup>A MTC to adjacent RNA polymerase II, and thereby delivering the m<sup>6</sup>A MTC to actively transcribed nascent RNAs to deposit m<sup>6</sup>A co-transcriptionally. In mouse embryonic stem cells, phenocopying Mettl14 silencing, H3K36me3 depletion also induces m<sup>6</sup>A reduction transcriptome-wide and in pluripotency transcripts, resulting in increased cell stemness. Collectively, our studies reveal the critical roles of H3K36me3 and METTL14 in determining precise and dynamic m<sup>6</sup>A deposition in mRNA, and uncover another layer of gene expression regulation involving crosstalk between histone modification and RNA methylation.</p></div></div><div id="body-1" class="tsec sec"><h2 class="headless nomenu"></h2><p id="P4" class="p p-first">Transcriptome-wide m<sup>6</sup>A mapping reveals the presence of m<sup>6</sup>A modification in approximately 7,000 mRNAs in both human and mouse transcriptomes and uncovers the consensus motif RRACH (R= G or A; H= A, C or U) in which A is converted to m<sup>6</sup>A<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621533" role="button" aria-expanded="false" aria-haspopup="true">13</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621546" role="button" aria-expanded="false" aria-haspopup="true">15</a></sup>. Despite the recognition of RRACH motifs by the METTL3-METTL14 methyltransferase complex <em>in vitro</em>, only a portion of them are methylated <em>in vivo</em>, which are typically enriched in the coding sequence (CDS) and 3’ untranslated region (UTR)<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621537" role="button" aria-expanded="false" aria-haspopup="true">13</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621552" role="button" aria-expanded="false" aria-haspopup="true">16</a></sup>. How individual transcripts and specific sites are selected for proper deposition of m<sup>6</sup>A modification remains elusive. Here we identified histone H3 trimethylation at lysine-36 (H3K36me3) as a determinant of m<sup>6</sup>A RNA modification.</p><p id="P5">By comparing the distribution of m<sup>6</sup>A in human transcriptome to that of different histone modifications, we found that 69.2% of m<sup>6</sup>A peaks overlapped with the H3K36me3 modification (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Extended Data Fig. 1a</span></a>). Distance analysis also showed an overrepresentation of H3K36me3 modifications, but not H3K27me3 (as a control), near m<sup>6</sup>A peaks (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Extended Data Fig. 1b</span></a>). Considering that H3K36me3 is the only known histone modification enriched in CDS and especially near the 3’ end<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621542" role="button" aria-expanded="false" aria-haspopup="true">17</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621547" role="button" aria-expanded="false" aria-haspopup="true">18</a></sup>, these results revealed a possible unappreciated connection between H3K36me3 and m<sup>6</sup>A RNA modification. Moreover, SET domain containing 2 (SETD2), the histone methyltransferase that specifically catalyzes H3K36me2 to H3K36me3<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621548" role="button" aria-expanded="false" aria-haspopup="true">19</a></sup>, showed a positive correlation in mRNA expression with the m<sup>6</sup>A MTC genes (i.e., <em>METTL3</em>, <em>METTL14</em> and <em>WTAP</em>) in most normal and cancer tissues/cell lines (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Extended Data Fig. 1c-f</span></a>). Suppression of cellular H3K36me3, either through knockdown/knockout of SETD2 or overexpression of KDM4A, one of the H3K36me3 demethylases<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621545" role="button" aria-expanded="false" aria-haspopup="true">20</a></sup>, resulted in a significant decrease of m<sup>6</sup>A level in total RNAs and/or poly(A) RNAs, comparable to the effects of depletion of each individual m<sup>6</sup>A MTC genes (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 2a-k</span></a>). This impact appears to be conserved in mammalian as heterozygous knockout of SETD2 in mouse c-kit<sup>+</sup> bone marrow cells also led to hypo-methylation of both H3K36me3 and m<sup>6</sup>A (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 2l-o</span></a>). In contrast, knockdown of SUV39H1, the H3K9me3 methyltransferase, did not affect m<sup>6</sup>A abundance (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 2p-r</span></a>). Gain-of-function experiments in SETD2-deficient A498 cells showed that the co-transcriptional catalytic activity of SETD2 is required for the restoration of H3K36me3 and m<sup>6</sup>A levels (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 2s-u</span></a>). On the other hand, however, silencing of m<sup>6</sup>A MTC genes did not affect cellular H3K36me3 level (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 2v</span></a>). Collectively, these results indicate a causal role of H3K36me3 on m<sup>6</sup>A abundance and suggest that loss of H3K36me3 impaired <em>de novo</em> m<sup>6</sup>A methylation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452392563152"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0001.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0001.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452392563152" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>H3K36me3 histone modification affects <em>de novo</em> m<sup>6</sup>A RNA methylation.</strong><p id="P31">(a) LC-MS/MS quantification of m<sup>6</sup>A abundance in poly(A) RNA from HepG2 cells with knockdown of SETD2 (shSETD2#1) or individual m<sup>6</sup>A MTC genes. Values are mean±SD of two independent experiments. (b) m<sup>6</sup>A level of ploy(A) RNA from A498 cells with ectopic expression of wild type (WT) or mutated (R1625C and ΔSRI) SETD2 or empty pEGFP vector. The m<sup>6</sup>A abundance was determined by dot blot and quantified (values are mean±SD). Note that R1625C-SETD2 is catalytically inactive<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621527" role="button" aria-expanded="false" aria-haspopup="true">28</a></sup>, whereas truncation of the small three-helix domain (ΔSRI-SETD2) impairs the direct binding between SETD2 and Pol II and the co-transcriptional installation of H3K36me3<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621540" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup>. (c) Scatter plot of m<sup>6</sup>A peaks in control (shCtrl) and SETD2 knockdown (shSETD2#1) HepG2 cells. The abundance of m<sup>6</sup>A peaks was calculated as enrichment fold (EF, IP/input) from 3 independent m<sup>6</sup>A-seq replications. Hypermethylated (hyper) peaks were in red, while hypomethylated (hypo) peaks were in green. (d) Distribution of H3K36me3 relative to the m<sup>6</sup>A peaks in HepG2 cells. (e and f) H3K36me3 (e) and m<sup>6</sup>A (f) levels in <em>MYC</em> CRD determined by ChIP-qPCR and gene-specific m<sup>6</sup>A assays in shSETD2#1 and control HepG2 cells. Values are mean±SD of three independent experiments. (g and h) H3K36me3 (left) and m<sup>6</sup>A (right) level on specific locus were detected in HEK293T cells co-transfected with dCas9-KDM4A (g) or dCas9-SETD2 (h) and specific sgRNAs or non-targeting control (sgNT) as indicated. Values are mean±SD of four independent experiments. Two-tailed student’s t-test was used to test difference in a, b, e, f, g and h; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001. Source data of a, b, e, f, g and h are in Supplementary Table 4.</p></div></div></div><p id="P6">Furthermore, we performed m<sup>6</sup>A-seq in parallel with H3K36me3 ChIP-seq upon SETD2 knockdown to evaluate the global effect of H3K36me3 on m<sup>6</sup>A deposition. ChIP-seq confirmed global reduction of H3K36me3, especially around stop codon, in SETD2 knockdown cells (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 3a-e</span></a>). Meanwhile, m<sup>6</sup>A-seq revealed a global hypomethylation of m<sup>6</sup>A in the transcriptome upon SETD2 depletion (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 3f-i</span></a>). Similarly, these m<sup>6</sup>A-hypo peaks were mostly found in protein-coding mRNAs and enriched near stop codons (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 3j-k</span></a>). Moreover, the most common m<sup>6</sup>A consensus sequence “GGAC” was enriched in these m<sup>6</sup>A-hypo peaks with the highest ratio (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 3l</span></a>). Consistent results were also obtained with m<sup>6</sup>A individual-nucleotide-resolution cross-linking and immunoprecipitation (miCLIP)<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621550" role="button" aria-expanded="false" aria-haspopup="true">15</a></sup> and subsequent sequencing (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig. 3m-q</span></a>).</p><p id="P7">Integrated analysis of ChIP-seq and m<sup>6</sup>A-seq data revealed that the chromosome position of H3K36me3 and m<sup>6</sup>A were well correlated (z-score=212.93, <em>P</em>=9.99e-4, Permutation test; <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4a</span></a>) and overlapped (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1d</span></a>). Notably, the distributions of H3K36me3 and m<sup>6</sup>A are mostly exclusive with that of heterochromatin markers H3K9me3 or H3K27me3 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4a-c</span></a>). Additional analyses revealed that the m<sup>6</sup>A peaks associated with H3K36me3 modifications (H3K36me3+) are enriched near stop codon, while those without such association (H3K36me3-) are enriched around start codon (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4d</span></a>), indicating that H3K36me3 plays a role in m<sup>6</sup>A determination preferentially at 3’ end of CDS. On the other hand, 63% of H3K36me3+ loci were negative for m<sup>6</sup>A modifications (m<sup>6</sup>A-) and were found to enrich in CDS and intronic regions rather than 3’ end (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4e</span></a>), consistent with the known functions of H3K36me3 in mediating deacetylation, preventing run-away transcription and regulating splicing<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621536" role="button" aria-expanded="false" aria-haspopup="true">21</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621535" role="button" aria-expanded="false" aria-haspopup="true">23</a></sup>. In loci where H3K36me3 and m<sup>6</sup>A are both present, 2,717 out of the 5,872 (46.3%) hypomethylated (FC&lt;0.5) H3K36me3 peaks upon <em>SETD2</em> knockdown also exhibit hypomethylation (FC&lt;0.5) of m<sup>6</sup>A, while only 427 (7.3%) hypomethylated H3K36me3 peaks exhibit m<sup>6</sup>A hypermethylation (FC&gt;2). Such co-regulation of H3K36me3 and m<sup>6</sup>A by SETD2 knockdown was confirmed in individual representative genes, such as <em>MYC</em>, <em>ACTB</em>, <em>KPNA6</em>, and <em>EEF1A</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1e-f</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4f-i</span></a>), similar to what was observed when KDM4A was overexpressed (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 4j-k</span></a>).</p><p id="P8">We further utilized CRISPR/dCas9-fusion, in which nuclease-deactivated Cas9 (dCas9) is guided by single guide RNAs (sgRNAs) and thus brings the fused protein to specific genomic regions for epigenetic modification<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621534" role="button" aria-expanded="false" aria-haspopup="true">24</a></sup>, to confirm the casual regulation of H3K36me3 on m<sup>6</sup>A on specific locus (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 5a-b</span></a>). As expected, co-expression of dCas9-KDM4A fusion protein with sgRNAs (sgMYC) targeting the coding region instability determinant (CRD) region of MYC, where high level of H3K36me3 was observed (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1e</span></a>), could partially remove H3K36me3 and subsequently impair m<sup>6</sup>A deposition on <em>MYC</em> mRNA (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1g</span></a>). On the other hand, co-expression of dCas9-SETD2 fusion with sgRNAs (sgGNG4) targeting the gene body of GNG4, where no detectable H3K36me3 and m<sup>6</sup>A modifications were found (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 5c</span></a>), increased H3K36me3 abundance in GNG4 gene body and m<sup>6</sup>A modification in the corresponding mRNA region (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1h</span></a>). Moreover, we also constructed an artificial fusion gene (MYC-GNG4) in which the 5’ UTR sequence of GNG4 was fused downstream of MYC CRD (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 5d</span></a>). We hypothesize that by fusing to MYC CRD, the H3K36me3 modification in GNG4 5’ UTR sequence will be increased due to the elongation of pol II and co-transcriptional deposition of H3K36me3<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621528" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup>. This was indeed the case, and more important, such fusion resulted in an elevated level of m<sup>6</sup>A modification that could be partially or completely abrogated when SETD2 was depleted (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended Data Fig. 5e</span></a>), further demonstrating that m<sup>6</sup>A modifications could be guided by H3K36me3.</p><p id="P9">We then compared the transcriptome-wide effect of SETD2 knockdown on m<sup>6</sup>A to that caused by knockdown of individual m<sup>6</sup>A MTC components (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). A given m<sup>6</sup>A site that displayed more than 1.5-fold reduction upon knockdown of a given m<sup>6</sup>A MTC gene was defined as the given MTC gene-responsive site. Among the SETD2-dependent m<sup>6</sup>A-hypo sites, 84% were responsive to (the depletion of) one or more individual MTC genes (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). SETD2 silencing led to a global m<sup>6</sup>A hypomethylation on METTL3-, METTL14-, or WTAP-responsive sites and particularly on the sites responsive to all three MTC genes (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>), and such reduction mainly occurred within CDS and 3’UTR (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>), as represented by <em>MYC</em> mRNA (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 6a</span></a>). Moreover, significant positive correlation (<em>r</em>=0.51–0.60, <em>P</em>&lt;2.22e-308) in the fold changes of m<sup>6</sup>A abundance was observed between SETD2 knockdown and m<sup>6</sup>A MTC gene knockdown (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2d</span></a>). RNA-seq revealed that knockdown of SETD2 or individual MTC genes tend to suppress gene expression (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 6b</span></a>), and their effects on the alterations of gene expression showed significant positive correlation (<em>r</em>=0.52–0.58, <em>P</em>&lt;2.22e-308, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 6c</span></a>). Further mRNA stability profiling and polysome profiling showed that SETD2 knockdown phenocopied METTL14 knockdown in affecting mRNA decay and translation (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 6d-h</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452393765856"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0002.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0002.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452393765856" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Transcriptome-wide cooperation of H3K36me3 and MTC in m<sup>6</sup>A regulation</strong><p id="P32">(a) Venn diagram of significantly hypomethylated peaks in HepG2 cells upon knockdown of SETD2 (shSETD2#1) or individual m<sup>6</sup>A MTC genes. (b) Cumulative curves showing m<sup>6</sup>A demethylation in m<sup>6</sup>A MTC gene-responsive peaks upon SETD2 silencing. <em>P</em> values were calculated using two-sided Wilcoxon and Mann-Whitney test. (c) Metagene profiles of m<sup>6</sup>A modifications in MTC gene-responsive peaks and non-responsive peaks. Note that only loci with H3K36me3 modification in the shCtrl cells were included in the analysis. (d) Correlation of fold-change (FC) in m<sup>6</sup>A abundance between SETD2 knockdown and individual MTC gene knockdown cells. Correlation coefficient (<em>r</em>) and <em>P</em> values were calculated by Pearson’s Correlation analysis.</p></div></div></div><p id="P10">Mechanistically, we found that depletion of H3K36me3 by SETD2 silencing impaired the interaction between m<sup>6</sup>A MTC proteins and their target mRNAs (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7a</span></a>), without affecting expression of individual m<sup>6</sup>A MTC genes or the interaction between METTL3 and METTL14 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7b-f</span></a>). These results imply that H3K36me3 plays a role in recruiting MTC to deposit m<sup>6</sup>A marks on RNAs. Indeed, the <em>in vivo</em> interaction between H3K36me3 and individual m<sup>6</sup>A MTC proteins was detected in a manner independent of RNA or DNA (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7g-h</span></a>). The binding appeared to be H3K36me3 specific as compared to H3K36me1 or H3K36me2 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7i</span></a>). The interaction between H3K36me3 and endogenous METTL3/METTL14 was also validated (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7j-k</span></a>). Furthermore, silencing of individual m<sup>6</sup>A MTC components <em>in cellulo</em> showed that depletion of METTL14, but not METTL3 or WTAP, impaired the interaction of MTC with H3K36me3 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7k</span></a>), suggesting that METTL14 is key to the binding of MTC with H3K36me3. Importantly, the effect of SETD2 knockdown on m<sup>6</sup>A reduction was largely impaired in the absence of METTL14 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7l</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452403246528"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0003.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0003.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452403246528" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>H3K36me3 is recognized by METTL14 and guides m<sup>6</sup>A modifications co-transcriptionally.</strong><p id="P33">(a) Co-immunoprecipitation and western blot in Hela cells with forced expression of HA-tagged m<sup>6</sup>A MTC genes. (b) Gel shift of H3K36me3 with recombinant human METTL3 or METTL14 in native SDS-PAGE gel. (c) Binding curves and calculated Kd of DyLight488-METTL14 with full length H3K36me3 or H3K9me3 peptide. Values are mean±SD of three independent experiments for H3K36me3 and two independent experiments for H3K9me3. (d) Distribution of METTL14 binding sites on chromatin identified by ChIP-seq. (e) Pearson correlation coefficients analysis of METTL14 ChIP-seq signals with genomic H3K36me3 features in non-overlapping, non-repetitive windows of different sizes along the genome. (f and g) Distribution of H3K36me3 (f) or m<sup>6</sup>A (g) relative to nearest METTL14 RNA binding sites identified by PAR-CLIP-seq. (h) Proposed model underlying the mechanism of H3K36me3-guided deposition of m<sup>6</sup>A methylation. Images in a and b were representative of three independent experiments. Source data of c are in Supplementary Table 4.</p></div></div></div><p id="P11">Two possible models underlying this interaction, indirect (through adaptors) or direct (Reading and Writing, R/W) binding (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8a</span></a>), were then proposed. The indirect model was excluded because no known H3K36me3 readers (adaptors) tested could affect binding of H3K36me3 with METTL14 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8b-d</span></a>). Instead, our <em>in vitro</em> (cell-free) pulldown assay and gel shift assay revealed a strong direct interaction between H3K36me3 and recombinant METTL14 protein (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3b</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8e-f</span></a>). Microscale thermophoresis (MST)<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621549" role="button" aria-expanded="false" aria-haspopup="true">26</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621539" role="button" aria-expanded="false" aria-haspopup="true">27</a></sup> was then performed to evaluate the binding affinity, which gave an estimated Kd value of 1.860±0.288 µM (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c</span></a>).</p><p id="P12">To determine which region in METTL14 contributes to its binding with H3K36me3, we conducted truncation analysis using immunoprecipitation (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8g</span></a>). Among the truncations, Δ186–456 is the shortest variant that could interact with H3K36me3 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8g-h</span></a>), suggesting that 1–185 amino acids are critical for the binding. Two fragments of METTL14, 138–143 and 153–161 amino acids, were further found to be important for the binding (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8i</span></a>). Functionally, only full length (FL), but not truncated variants of METTL14 (Δ138–143 or Δ153–161), could restore the m<sup>6</sup>A abundance in METTL14 inducible knockout cells (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8j</span></a>).</p><p id="P13">Because METTL14 interacts with H3K36me3, we speculate that it also binds chromatin. Indeed, ChIP-seq with HA-tagged METTL14 using an HA antibody identified 1,973 peaks in HepG2 cells, among which 1,278 (65%) peaks were located in gene body (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3d</span></a>). We further revealed genome-wide significant correlation between chromatin binding of METTL14 and H3K36me3, but not H3K27me3 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3e</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8k</span></a>). In addition, PAR-CLIP-seq revealed 5,981 potential METTL14 binding sites on RNA (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8l-n</span></a>). Distance analysis showed that METTL14 binding sites on RNA overlapped well with H3K36me3 and m<sup>6</sup>A modification (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3f-g</span></a>), supporting our proposed function of METTL14 in linking H3K36me3 with m<sup>6</sup>A deposition.</p><p id="P14">In line with previous reports that H3K36me3 was established by SETD2 when associated with Pol II during transcription elongation<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621543" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup>, we demonstrated the association of m<sup>6</sup>A MTC proteins with Pol II (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3a</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 7h</span></a>), and specifically, the elongating form of Pol II (pSer2) (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8o</span></a>), and found that METTL14 co-localizes with H3K36me3 and pSer2-Pol II in the nucleus (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8p</span></a>), supporting a co-transcriptional regulation of H3K36me3 on m<sup>6</sup>A. In addition, newly transcribed RNAs associated with chromatin exhibited higher abundance of m<sup>6</sup>A modification than nucleoplasmic or cytoplasmic RNAs (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8q</span></a>). To better understand this process, we treated cells with a Pol II inhibitor 5,6-Dichlorobenzimidazole 1-β-D-ribofuranoside (DRB) for a short time. We found DRB decreased m<sup>6</sup>A abundance without affecting H3K36me3 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8r-t</span></a>). Notably, the interaction of METTL14 with H3K36me3, chromatin, or METTL3, was not interrupted by DRB treatment (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended Data Fig. 8u-v</span></a>). Therefore, our results suggest a working model in which METTL14, a critical component of the m<sup>6</sup>A MTC, directly reads H3K36me3 in a Pol II independent manner, which allows the m<sup>6</sup>A MTC to methylate adjacent nascent RNAs during Pol II-mediated transcription elongation (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3h</span></a>).</p><p id="P15">Dynamic regulation of m<sup>6</sup>A modifications is crucial for embryonic stem cells (ESCs) self-renewal and differentiation<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621538" role="button" aria-expanded="false" aria-haspopup="true">5</a>-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621551" role="button" aria-expanded="false" aria-haspopup="true">7</a></sup>. To decipher whether the regulation of H3K36me3 on m<sup>6</sup>A RNA modification influences cell differentiation, we utilized the mouse ESC (mESC) model with doxycycline (Dox)-induced SETD2 knockdown for induction of <em>in vitro</em> differentiation through leukemia inhibitory factor (LIF) deprivation (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9a-b</span></a>). Upon Dox treatment, mESCs expressed higher levels of pluripotency factors, including OCT4, SOX2 and NANOG (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a</span></a>), and showed stronger staining of cell surface SSEA1 and nuclear OCT4 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9c</span></a>), suggesting a more primitive state. On the other hand, the increase of endoderm markers (i.e., GATA4, GATA6, DAB2, and SOX7) during differentiation was attenuated in the presence of Dox (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9d</span></a>). The effects of SETD2 knockdown on mESC differentiation were similar to those caused by METTL14 knockdown (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9e-g</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452390433824"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0004.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0004.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452390433824" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Loss of H3K36me3 reduces m<sup>6</sup>A in mouse embryonic cells and delays <em>in vitro</em> differentiation.</strong><p id="P34">(a) Western blot showing increase of pluripotency factors in mESCs with Dox-induced SETD2 knockdown. (b) Distribution of genes with more than 1.5 fold changes (<em>P</em>&lt;0.05) in both m<sup>6</sup>A level and H3K36me3 level in SETD2 knockdown mESCs compared to control cells. (c and d) Distribution of H3K36me3 peaks (c) and m<sup>6</sup>A peaks (d) across <em>SOX2</em> mRNA transcript. (e) m<sup>6</sup>A abundance of individual sites in mRNA transcripts of pluripotency factors during differentiation was determined by m<sup>6</sup>A-IP and qRT-PCR. Values are mean±SD of three independent experiments; two-tailed student’s t-test; ***, <em>P</em>&lt;0.001. Images were representative of three independent experiments. Source data of e are in Supplementary Table 4.</p></div></div></div><p id="P16">As expected, global H3K36me3 and cellular m<sup>6</sup>A levels were substantially reduced in mESCs upon SETD2 knockdown (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9h-m</span></a>). Noticeably, most of the genes with H3K36me3 hypomethylation showed significantly reduced m<sup>6</sup>A levels (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4b</span></a>; <em>P</em>&lt;2.22e-16 by Pearson’s Chi-squared test), as represented by <em>SOX2</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4c-d</span></a>). Functional analysis of gene expression revealed the enrichment of stem cell maintenance genes in downregulated genes during LIF withdrawal and in upregulated genes upon SETD2 silencing (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9n</span></a>). In representative pluripotency genes with significant decrease of H3K36me3 upon Setd2 knockdown, an increase of m<sup>6</sup>A level was observed during differentiation, which was prevented by Dox-induced SETD2 knockdown (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4e</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9o</span></a>). Consistent with the reported role of m<sup>6</sup>A in destabilizing mRNAs (especially in ESCs)<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_753621529" role="button" aria-expanded="false" aria-haspopup="true">2</a></sup>, these pluripotency genes showed a dramatic decrease in mRNA level during differentiation, which could be partially rescued by SETD2 knockdown (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 9p</span></a>). These results provide evidences that loss of H3K36me3 reduces m<sup>6</sup>A in mESCs and delays <em>in vitro</em> differentiation. Furthermore, knockdown of METTL14 in SETD2-silenced cells could moderately further delay cell differentiation (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F14/" target="figure" class="fig-table-link figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 10a-c</span></a>). On the other hand, overexpression of METTL14 promoted differentiation in control cells, and could partially rescue differentiation inhibition in SETD2 knockdown cells (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F14/" target="figure" class="fig-table-link figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 10d-f</span></a>). Collectively, our results reveal that SETD2 and H3K36me3 inhibit pluripotency and promote differentiation of mESCs at least in part through regulation of m<sup>6</sup>A on several critical pluripotency genes.</p><p id="P17" class="p">In summary, our results uncover a mechanism underlying the dynamic, precision and co-transcriptional deposition of m<sup>6</sup>A modification in mammalian transcriptomes, explaining its prevalent occurrence to CDS and 3’UTR. The identification of the histone mark H3K36me3 as a determinant of m<sup>6</sup>A RNA modification via METTL14 reveals a previously unknown cross-talk between histone modification and RNA methylation and therefore adds another layer of complexity to gene expression control in normal and pathological biological processes.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Extended Data</h2><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F5"><h3>Extended Data Figure 1.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452406512208"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0005.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0005.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452406512208" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F5/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overlapping of H3K36me3 with m<sup>6</sup>A and co-expression of their writer genes.</strong><p id="P18">(a) Overlaps of m<sup>6</sup>A sites with histone modification sites in human genome. Histogram showed the observed and expected percentages of m<sup>6</sup>A sites that overlap with various histone modification sites; the ENCODE ChIP-seq data and m<sup>6</sup>A-seq data (<a id="__tag_753621553" class="tag_hotlink" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37003" ref="/geo/query/acc.cgi?acc=GSE37003"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE37003","term_id":"37003"}}</span>GSE37003</a>) obtained from HepG2 cells were used for the analyses. (b) Distance of histone modifications to the nearest m<sup>6</sup>A peaks. Input and H3K27me3 were shown as negative controls. (c-e) Correlation of SETD2 with individual m<sup>6</sup>A MTC genes (i.e., METTL3, METTL14, WTAP) in expression in normal tissues (c), cancer samples (d), and cell lines (e), based on the data from Genotype Tissue Expression (GTEx), The Cancer Genome Altlas (TCGA), and Cancer Cell Line Encyclopedia (CCLE) databases, respectively. Note that every dot represents one tissue type (c) or one cancer type (d). (f) Co-expression analysis of SETD2 and individual m<sup>6</sup>A MTC genes in 33 cancer cell lines based on our qPCR data. The value of each gene represents relative expression as normalized to HEK293T cells. Correlation coefficient (<em>r</em>) and <em>P</em> value were calculated by Pearson’s Correlation analysis in c, d, e and f.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F6"><h3>Extended Data Figure 2.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452390098080"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0006.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0006.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452390098080" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F6/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Cellular m<sup>6</sup>A level was regulated by the H3K36me3 methyltransferase SETD2 and the demethylase KDM4A.</strong><p id="P19">(a) Western blot showing knockdown efficiency of SETD2 and decrease of H3K36me3 in HepG2 and Hela cells by shSETD2#1. GAPDH and H3 serve as loading controls. (b) LC-MS/MS quantification of m<sup>6</sup>A abundance in poly(A) RNA from Hela cells with knockdown of SETD2 (shSETD2#1) or individual m<sup>6</sup>A MTC genes compared to control cells. Values are mean±SD of two independent experiments. (c) Knockdown efficiency of shRNAs was verified by qPCR assays. Values are mean±SD of three independent experiments. (d) Dot blot (left) and quantification (right, values are mean±SD) of m<sup>6</sup>A in total RNAs from control cells or cells with knockdown of SETD2 or individual m<sup>6</sup>A MTC genes. MB: methylene blue, serving as loading control. (e) Knockdown of SETD2 by two shRNAs were verified by western blot in HepG2 cells. (f) Dot blot (right) and quantification (left, values are mean±SD) of m<sup>6</sup>A in total RNAs from SETD2 knockdown HepG2 cells. (g) Western blot showing depletion of SETD2 and H3K36me3 in wild type (WT) or SETD2 knockout (KO) Hela cells. (h) LC-MS/MS quantification of m<sup>6</sup>A in poly(A) RNA from WT or SETD2 KO Hela cells. Values are mean±SD of two independent experiments. (i) Dot blot (right) and quantification (left, values are mean±SD) of m<sup>6</sup>A in total RNA from WT or SETD2 KO Hela cells. (j) Western blot confirmed the overexpression of HA-tagged KDM4A and decrease of H3K36me3 in HEK293T cells. (k) Quantification (left, values are mean±SD) and dot blot (right) showed hypomethylation of m<sup>6</sup>A in poly(A) RNAs from HEK293T cells with KDM4A overexpression compared to control cells. (l) Genotyping of SETD2 heterozygous knockout (+/−) mice. c-kit+ bone marrow (BM) cells were used. (m) Western blot showing H3K36me3 level in c-kit+ BM cells from SETD2 heterozygous knockout (+/−) mice. (n and o) LC-MS/MS quantification (n) and dot blot analysis (o, values are mean±SD on left) of m<sup>6</sup>A in poly(A) RNA from c-kit+ BM cells of wild type (+/+) and Setd2+/− mice. Values are mean±SD of two technical replications and 3 mice were used for each group in n. (p) Western blot showing expression of SUV39H1 and H3K9me3 in Hela cells transfected with siRNA against SUV39H1 (siSUV39H1) or negative control (siNC). (q and r) Dot blot and quantification (values are mean±SD on left) of m<sup>6</sup>A in total RNA (q) or poly(A) RNA (r) from Hela cells transfected with siSUV39H1 or siNC. (s) A498 cells were transfected with GFP-fused wild type or mutated SETD2, and were selected for GFP positive cells after 48 hours by cell sorting. The elevation of SETD2 mRNA level in SETD2-overexpressed cells was confirmed by qPCR. Values are mean±SD of three independent experiments. (t) Western blot showing H3K36me3 level in A498 cells with (wild-type or mutant) or without SETD2 overexpression. HEK293T served as a positive control. (u) Dot blot detecting m<sup>6</sup>A in poly(A) RNAs from A498 cells with (wild-type or mutant) or without SETD2 overexpression. (v) Western blot showing that silencing of individual m<sup>6</sup>A MTC genes did not affect cellular H3K36me3 level. Two-tailed student’s t-test was used in b, c, d, f, h, i, k, n, o, q, r and s; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001; N.S., non-significant. Images in a, d, g, and i were representative of three independent experiments, e, f, j, k, m, o, p, q, r, t, u and v were representative of two independent experiments. Source data of b, c, d, f, h, i, k, n, o, q, r and s are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F7"><h3>Extended Data Figure 3.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452390459456"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0007.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0007.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452390459456" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F7/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Reprogramming of H3K36me3 and m<sup>6</sup>A upon silencing of SETD2.</strong><p id="P20">(a) Histogram showing H3K36me3 peak numbers in HepG2 cells with (shSETD2#1) or without (shCtrl) SETD2 knockdown. (b) Cumulative curves and box plot showing reduction of H3K36me3 in SETD2 knockdown HepG2 cells relative to control cells<em>. P</em> values were calculated using two-sided Wilcoxon and Mann-Whitney test. For the box plot, top of upper whisker is 95<sup>th</sup> percentile (shCtrl=3.436, shSETD2#1=2.073), top of the box is 75<sup>th</sup> percentile (shCtrl=1.898, shSETD2#1=1.035), line through the box is median (shCtrl=1.358, shSETD2#1=0.645), bottom of the box is 25<sup>th</sup> percentile (shCtrl=0.868, shSETD2#1=0.339), and bottom of the lower whisker is 5<sup>th</sup> percentile (shCtrl=0.001, shSETD2#1=0.001). (c) Circos plot showing genome-wide decrease of H3K36me3 in SETD2 knockdown HepG2 cells compared to control cells. (d) The proportion (pie chart on top) and enrichment (histogram on bottom) of hypomethylated H3K36me3 peak distribution in the promoter, 5’UTR, CDS, stop codon, or 3’UTR region in SETD2 knockdown HepG2 cells. The enrichment was determined by the proportion of H3K36me3 peaks normalized by the length of the region. (e) Metagene profiles of H3K36me3 in SETD2 knockdown and control HepG2 cells. (f) Cumulative curves of m<sup>6</sup>A abundance [i.e., log<sub>2</sub>(m<sup>6</sup>A EF+1)] in control and SETD2 knockdown (shSETD2#1) HepG2 cells. Abundance of m<sup>6</sup>A-IP was normalized to input when calculating the enrichment fold. <em>P</em> value was calculated using two-sided Wilcoxon and Mann-Whitney test. (g) Scatter plot of m<sup>6</sup>A peaks in control (shCtrl) and SETD2 knockdown (shSETD2#2) HepG2 cells. The abundance of m<sup>6</sup>A peaks was calculated as enrichment fold (EF, IP/input) from 3 independent m<sup>6</sup>A-seq replications. Hypermethylated (hyper) peaks were in red, while hypomethylated (hypo) peaks were in green. (h) Cumulative curves of m<sup>6</sup>A abundance [i.e., log<sub>2</sub>(enrichment fold+1)] in control and SETD2 knockdown (shSETD2#2) HepG2 cells. <em>P</em> value was calculated using two-sided Wilcoxon and Mann-Whitney test. (i) Circos plot showing the distribution of hypermethylated and hypomethylated m<sup>6</sup>A peaks in human transcriptome (left) and chromosome 5 (right) upon knockdown of SETD2 or individual m<sup>6</sup>A MTC genes. (j) Pie charts showing the percentages of hypomethylated m<sup>6</sup>A peaks distribution in various RNA species by SETD2 knockdown in HepG2 cells. (k) The proportion (pie chart on top) and enrichment (histogram on bottom) of hypomethylated m<sup>6</sup>A peak distribution in the 5’UTR, CDS, stop codon, or 3’UTR region in SETD2 knockdown HepG2 cells. The enrichment was determined by the proportion of m<sup>6</sup>A peaks normalized by the length of the region. (l) HOMER motif analysis revealed conserved motifs enriched in hypomethylated peaks. <em>P</em> values were calculated by HOMER algorithm. (m) Numbers of m<sup>6</sup>A sites in control (shCtrl) and SETD2 knockdown (sh#1 or sh#2) HepG2 cells identified by miCLIP. (n) HOMER motif analysis revealed conserved motifs of C to T mutations or truncations identified by miCLIP. <em>P</em> values were calculated by HOMER algorithm. (o) The proportion (pie chart on top) and enrichment (histogram on bottom) of miCLIP-identified m<sup>6</sup>A distribution in gene body regions. (p) Percentages of various RNA species containing miCLIP-identified m<sup>6</sup>A residues. (q) Metagene profiling of miCLIP-identified m<sup>6</sup>A residues in control and SETD2 knockdown cells.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F8"><h3>Extended Data Figure 4.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452402772256"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0008.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0008.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0008.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452402772256" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F8/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-/Transcriptome-wide and locus-specific co-regulation of H3K36me3 and m<sup>6</sup>A.</strong><p id="P21">(a) Circos plot showing global distribution of H3K36me3 and m<sup>6</sup>A, as well as H3K9me3, in human genome (left) and chromosome 5 (right) of HepG2 cells. (b) Distribution of H3K9me3 relative to nearest m<sup>6</sup>A sites in HepG2 cells. (c) Percentages and peak numbers of H3K36me3 and m<sup>6</sup>A in H3K9me3 negative (H3K9me3-), H3K9me3 positive (H3K9me3+), H3K27me3 negative (H3K27me3-), or H3K27me3 positive (H3K27me3+) regions in HepG2 cells. (d) Distinct patterns of m<sup>6</sup>A metagene at H3K36me3 positive (H3K36me3+) or negative (H3K36me3-) sites. (e) Distribution of H3K36me3 peaks with m<sup>6</sup>A (m<sup>6</sup>A+) or without m<sup>6</sup>A (m<sup>6</sup>A-) modifications in gene body and intergenic regions. (f and g) Quantification of H3K36me3 (f) and m<sup>6</sup>A (g) level in representative genes as determined by ChIP-qPCR and gene specific m<sup>6</sup>A-qPCR assays in shSETD2#1 and control (shCtrl) HepG2 cells. (h and i) Quantification of H3K36me3 (h) and m<sup>6</sup>A (i) level in representative genes as determined by ChIP-qPCR and gene specific m<sup>6</sup>A-qPCR assays in shSETD2#2 and control (shCtrl) HepG2 cells. (j and k) Quantification of H3K36me3 (j) and m<sup>6</sup>A (k) level in representative genes as determined by ChIP-qPCR and gene specific m<sup>6</sup>A-qPCR assays in KDM4A-overexpressed and control HEK293T cells. Values are mean±SD of three independent experiments in f, g, h, i, j and k; two-tailed student’s t-test; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001. Source data of f, g, h, i, j and k are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F9"><h3>Extended Data Figure 5.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452392717360"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0009.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0009.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0009.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452392717360" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F9/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Locus-specific regulation of m<sup>6</sup>A by H3K36me3.</strong><p id="P22">(a, b) Schematic showing the epigenetic editing of H3K36me3 by dCas9-KDM4A on MYC-CRD (a) or by dCas-SETD2 on GNG4 (b). (c) Distribution of H3K36me3 and m<sup>6</sup>A peaks across <em>GNG4</em> mRNA transcript. (d) Schematic showing the endogenous GNG4 and artificial MYC-GNG4 fusion gene. The primers used to distinguish endogenous GNG4 (GNG4-5’UTR), MYC-GNG4 fusion sites (MYC-GNG4) or GNG4-vector fusion sites (GNG4-pcDNA) were indicated by black, orange, or blue arrows, respectively. (e) Quantification of H3K36me3 (top) and m<sup>6</sup>A (bottom) in endogenous GNG4 or MYC-GNG4 fusion gene by ChIP-qPCR or gene specific m<sup>6</sup>A assay in control or SETD2 knockdown cells. Values are mean±SD of five independent experiments. Two-tailed student’s t-test was used; ***, <em>P</em>&lt;0.001; N.S., non-significant. Source data of e are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F10"><h3>Extended Data Figure 6.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452403718816"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0010.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0010.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0010.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452403718816" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F10/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Impact of SETD2 knockdown on gene expression, mRNA stability and translation.</strong><p id="P23">(a) Distribution of H3K36me3 and m<sup>6</sup>A peaks across <em>MYC</em> mRNA transcript. CRD: coding region instability determinant. (b) Histogram showing significant upregulated (up, log<sub>2</sub>FC&gt;0.585, <em>P</em>&lt;0.05) and downregulated (down, log<sub>2</sub>FC&lt;−0.585, <em>P</em>&lt;0.05) genes in SETD2 or individual MTC gene knockdown HepG2 cells. (c) Correlation of fold-change (FC) in gene expression between SETD2 knockdown and individual MTC gene knockdown cells. (d) Cumulative curves of mRNA half-life in SETD2 or METTL14 knockdown or control cells. <em>P</em> value was calculated using two-sided Wilcoxon and Mann-Whitney test. (e) Decay curves and half-life (T<sub>1/2</sub>) of <em>MYC</em> mRNA in SETD2 or METTL14 knockdown or control cells were derived from transcriptome-wide mRNA stability profiling. Values are mean±SD of two independent experiments. (f) Correlation of changes in mRNA half-life between SETD2 knockdown and METTL14 knockdown cells. (g) Histograms showing genes with significantly increase (up, log<sub>2</sub>FC&gt;0.585, <em>P</em>&lt;0.05) or decrease of translation efficiency (down, log<sub>2</sub>FC&lt;−0.585, <em>P</em>&lt;0.05) in SETD2 or METTL14 knockdown HepG2 cells. (h) Correlation of changes in translation efficiency between SETD2 knockdown and METTL14 knockdown cells. Correlation coefficient (<em>r</em>) and <em>P</em> values were calculated by Pearson’s Correlation analysis in c, f and h. Source data of e are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F11"><h3>Extended Data Figure 7.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452401504016"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0011.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0011.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0011.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452401504016" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F11/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Mechanism of H3K36me3-dependent m<sup>6</sup>A modification.</strong><p id="P24">(a) Binding of METTL3 (MTL3, upper panel) and METTL14 (MTL14, lower panel) to target mRNAs was determined by CLIP-qPCR assays. (b) qPCR data showing that knockdown of SETD2 in HepG2 and Hela cells did not downregulate the expression of m<sup>6</sup>A MTC genes. (c) qPCR data showing that overexpression of SETD2 in A498 cells did not significantly change the expression of m<sup>6</sup>A MTC genes. (d) qPCR showing that overexpression of KDM4A in HEK293T cells did not affect the expression of individual m<sup>6</sup>A MTC genes i. GAPDH were used as reference gene when normalizing qPCR data in b, c and d. (e) Protein level of individual MTC in SETD2 silencing, SETD2 overexpression (OE), or KDM4A overexpression cells was determined by western blot. (f) Interaction of METTL3 and METTL14 in SETD2 knockdown or control cells with forced expression of HA-METTL3. (g) Western blot showing forced expression of METTL3, METTL14, and WTAP in Hela cells. (h) Co-IP of HA-tagged individual m<sup>6</sup>A MTC proteins in Hela cell lysates with or without pretreatment with DNase (DN, 100 U/ml) or RNase (RN, 2 ug/ml) for 1-hour at room temperature. (i) Co-IP of HA-tagged METTL14 showed no interaction between METTL14 and mono- or di-methylation of H3K36 (H3K36me1 and H3K36me2) in Hela cells. (j) Co-IP of endogenous METTL3 (M13) or METTL14 (M14) in Hela cells. The interaction between METTL3 or METTL14 with H3K36me3 was detected by western blot. (k) Co-IP of endogenous H3K36me3 in HepG2 cells without or with knockdown of individual m<sup>6</sup>A MTC genes. (l) The m<sup>6</sup>A/A ratio was quantified by LC-MS/MS in HepG2 cells with knockdown of SETD2 or/and METTL14. Values are mean±SD of two independent experiments in b and l and of three independent experiments in a, c and d; two-tailed student’s t-test was used to test difference; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001. Images in e, f, g, h, i, j and k were representative of three independent experiments. Source data of a, b, c, d and l are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F12"><h3>Extended Data Figure 8.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452386403840"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0012.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0012.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0012.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452386403840" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F12/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>METTL14 binds to H3K36me3 <em>in vitro</em> and <em>in vivo</em>.</strong><p id="P25">(a) Schematic of the indirect or direct models of H3K36me3 recruiting MTC. “Adaptor” model (left) refers to indirect interaction of m<sup>6</sup>A MTC with H3K36me3 through known H3K36me3 binding proteins (adaptors). “Reading and Writing” (R/W) model is shown on right, which proposes that m<sup>6</sup>A writer complex functions as reader of H3K36me3 (green dot) and is recruited to chromatin to catalyze m<sup>6</sup>A methylations (yellow dots) in newly synthesized RNAs. (b) Potential association of METTL3 or METTL14 with known H3K36me3 readers was examined by co-IP in Hela cells with forced expression of HA-tagged METTL3 or METTL14. (c) Western blot confirmed knockdown efficiency of MSH2 by siRNA. (d) Western blot showing that MSH2 siRNA did not affect the interaction between METTL14 and H3K36me3. (e) The direct binding of Flag-tagged recombinant human METTL3 or METTL14 proteins with histone 3 peptides with (me3+) or without (me3-) K36me3 modifications was examined by <em>in vitro</em> pull-down assays. (f) Gel shift of H3K9me3 or unmethylated histone H3 with recombinant human METTL3 or METTL14 in native SDS-PAGE gel. (g) Schematic of full-length (FL) METTL14 and its truncations was shown on top. Co-IP coupled with western blot showing the interaction of ectopically expressed FL or truncated METTL14 with H3K36me3 in Hela cells was shown on the bottom. (h) Co-IP coupled with western blot showed the interaction of ectopically expressed truncated (Δ186-456 or Δ117-456) METTL14 with H3K36me3 in HepG2 cells. (i) Co-IP coupled with western blot showed the interaction of ectopically expressed full length (FL) or truncated (Δ138-143 or Δ153-161) METTL14 with H3K36me3 in HepG2 cells. (j) m<sup>6</sup>A Dot blot assays in METTL14 inducible knockout cells (sgMETTL14) transduced with different variants of METTL14. Images of m<sup>6</sup>A and methylene blue blots were shown on right, while the quantification of m<sup>6</sup>A was shown on left (values are mean±SD). (k) Pearson correlation coefficients of METTL14 ChIP-seq peaks with genomic H3K27me3 features in non-overlapping, non-repetitive windows of different sizes along the genome. (l) Percentages of various RNA species containing METTL14 binding sites detected by PAR-CLIP-seq. (m) The proportion (pie chart on top) and enrichment (histogram on bottom) of METTL14 CLIP binding sites distribution in gene body regions. (n) HOMER motif analysis of T to C mutations or truncations identified by METTL14 PAR-CLIP. <em>P</em> value was calculated by HOMER algorithm. (o) Co-IP showed that METTL14 bound to Ser2 phosphorylated polymerase II (Pol II pSer2) in Hela cells. (p) Co-localization of METTL14 with H3K36me3 (upper panels) or active transcription polymerase II (pSer2) (lower panels) in nuclei of HepG2 cells. Scale bar =10 μm. (q) Dot blot (right) and quantification (left, values are mean±SD) showing enrichment of m<sup>6</sup>A in chromatin bound RNAs compared to that in RNAs from other cell fractions. (r) Western blot showing DRB treatment (100µM for 3 hours) did not affect H3K36me3 level in Hela cells. DMSO, was used as a vehicle control. (s and t) Dot blot and quantification (values are mean±SD on left) showing the m<sup>6</sup>A abundance of total RNA (s) or poly(A) RNA (t) was reduced upon DRB treatment (at 100µM for 3 hours) in Hela cells. (u) Co-IP showing association of METTL14 with H3K36me3, METTL3 or Pol II CTD with or without DRB treatment. (v) Distribution of METTL14 binding sites on chromatin in DRB treated cells. Images in b, c, d, e, f, g, h, i, j, o, q, r, s, t and u were representative of three independent experiments. Two-tailed student’s t-test was used to test difference in j, q, s and t; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001; N.S., non-significant. Source data of j, q, s and t are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F13"><h3>Extended Data Figure 9.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452385364976"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0013.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0013.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0013.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452385364976" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F13/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Silencing of SETD2 demethylates H3K36me3 and m<sup>6</sup>A of pluripotency factors and delays the <em>in vitro</em> differentiation of mESCs.</strong><p id="P26">(a) qPCR showing remarkable downregulation of SETD2, but not m<sup>6</sup>A MTC genes, in doxycycline (Dox)-induced Setd2 knockdown mESCs. (b) Western blot showing reduction of Setd2 and decrease of H3K36me3 in doxycycline (Dox)-induced SETD2 knockdown mESCs after 48 hours of Dox treatment. (c) Immunofluorescence showing expression of Stage-specific embryonic antigen-1 (SSEA-1, left) and OCT4 (right) in control (-Dox) and SETD2 knockdown (+Dox) mESCs after 3 days of LIF withdrawal. Scale bar =10 μm. (d) Expression level changes of endoderm differentiation markers during <em>in vitro</em> differentiation of mESCs. (e) Silencing of METTL14 by shRNAs in mESCs as determined by qRT-PCR. (f) Silencing of METTL14 promotes the expression of pluripotency markers in mESCs. HSP90 serves as a loading control. (g) Silencing of METTL14 delayed emerging of endoderm markers in mESCs as determined by qPCR assays. (h) Numbers of H3K36me3 peaks identified in mESCs with or without Dox-induced SETD2 knockdown by ChIP-seq. (i) ChIP-seq revealed global hypomethylation of H3K36me3 in mESCs with doxycyclin treatment as shown by histogram distribution and box plots. For the box plot, top of upper whisker is 95<sup>th</sup> percentile (shCtrl=2.519, shSETD2=1.081), top of the box is 75<sup>th</sup> percentile (shCtrl=1.281, shSETD2=0.527), line through the box is median (shCtrl=0.886, shSETD2=0.312), bottom of the box is 25<sup>th</sup> percentile (shCtrl=0.454, shSETD2=0.155), and bottom of the lower whisker is 5<sup>th</sup> percentile (shCtrl=0.000, shSETD2=0.000). <em>P</em> value was calculated using two-sided Wilcoxon and Mann-Whitney test. (j) Dot blot (left) and quantification (right, values are mean±SD) revealed reduction of cellular m<sup>6</sup>A in mESCs by doxycycline-induced knockdown of Setd2. (k) m<sup>6</sup>A-seq revealed global hypomethylation of m<sup>6</sup>A in mESCs with doxycyclin treatment as shown by histogram distribution and box plots. For the box plot, top of upper whisker is 95<sup>th</sup> percentile (shCtrl=18.500, shSETD2=10.200), top of the box is 75<sup>th</sup> percentile (shCtrl=9.530, shSETD2=5.360), line through the box is median (shCtrl=5.480, shSETD2=3.700), bottom of the box is 25<sup>th</sup> percentile (shCtrl=3.530, shSETD2=2.110), and bottom of the lower whisker is 5<sup>th</sup> percentile (shCtrl=2.000, shSETD2=0.000). <em>P</em> value was calculated using two-sided Wilcoxon and Mann-Whitney test. (l) HOMER motif analysis revealed conserved motifs enriched in hypomethylated peaks. <em>P</em> value was calculated by HOMER algorithm. (m) The enrichment of hypomethylated m<sup>6</sup>A peak distribution. The enrichment fold was determined by the proportion of m<sup>6</sup>A peaks normalized by the length of the region. (n) Gene set enrichment analysis (GSEA)<sup>58</sup> of downregulated genes upon differentiation (D0 -&gt; D6 down) and upregulated genes upon SETD2 silencing (-Dox -&gt; +Dox up). (o) Determination of H3K36me3 abundance in pluripotency factors by ChIP-qPCR. (p) mRNA levels of pluripotency factors in mESCs as examined by qPCR. Values are mean±SD of two independent experiment in a, or three independent experiments in d, e, g, j, o and p; two-tailed student’s t-test; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001. Images in b, f and j were representative of three independent experiments. Source data of a, d, e, g, j, o and p are in Supplementary Table 4.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="F14"><h3>Extended Data Figure 10.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140452389593104"><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6438714_nihms-1521376-f0014.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-1521376-f0014.jpg" title="Click on image to zoom" class="tileshop" src="./pubmed_files/nihms-1521376-f0014.jpg"></a></div><div class="largeobj-link align_right" id="largeobj_idm140452389593104" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/figure/F14/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Impact of SETD2 and METTL14 in mESCs differentiation.</strong><p id="P27">(a and b) qPCR detecting the expression of <em>SETD2</em> (a) and <em>METTL14</em> (b) in mESCs. (c) The fold changes (D6 vs. D0) of gene expression of pluripotency factors and differentiation markers during <em>in vitro</em> differentiation of mESC with SETD2 and/or METTL14 knockdown. (d and e) qPCR detecting the expression of <em>SETD2</em> (d) and <em>METTL14</em> (e) showed the knockdown of SETD2 and overexpression of METTL14 in mESCs. (f) The fold changes (D6 vs. D0) of gene expression of pluripotency factors and differentiation markers during <em>in vitro</em> differentiation of mESC with SETD2 knockdown and/or METTL14 overexpression. Values are mean±SD of three independent experiments in a, b, c, d, e and f; two-tailed student’s t-test; *, <em>P</em>&lt;0.05; **, <em>P</em>&lt;0.01; ***, <em>P</em>&lt;0.001; N.S., non-significant. Source data of a, b, c, d, e, and f are in Supplementary Table 4.</p></div></div></div><div id="SM1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SM1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d36e1877"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(16K, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d36e1881"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(46M, pdf)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d36e1885"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/bin/NIHMS1521376-supplement-3.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(56K, docx)</sup></div></div></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Acknowledgements:</h2><div class="sec"><p id="P28">This work was supported in part by the National Institutes of Health (NIH) Grants R01 CA214965 (J.C.), R01 CA211614 (J.C.), R01 CA178454 (J.C.), R01 CA182528 (J.C.), RM1 HG008935 (C.H.), R21 CA187276 (G.H.), R01 CA163493 (J.G.), R35CA197628 (M.M.), U10CA180827 (M.M.), R01CA137060 (M.M.), R01CA157644 (M.M.), R01CA172558 (M.M.) and R01CA213138 (M.M.), and grants 2017YFA0504400 (J.Y.), 91440110 (J.Y.) and 31671349 (L.Q.) from National Nature Science Foundation of China, and Cancer Center Support Grant (P30CA33572) from City of Hope National Medical Center. J.C. is a Leukemia &amp; Lymphoma Society (LLS) Scholar. C.H. is an investigator of the Howard Hughes Medical Institute (HHMI). M.M. is a Howard Hughes Medical Institute (HHMI) Faculty Scholar. B.S.Z. is an HHMI International Student Research Fellow. F.A. was supported by a Deutsche Forschungsgemeinschaft (DFG) fellowship (AU 525/1–1).</p></div></div><div id="fn-group-1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="fn-group-1title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P29" class="p p-first">COMPETING FINANCIAL INTERESTS:</p><p id="P30" class="p p-last">C.H. is a scientific founder and a scientific advisor board member of Accent Therapeutics, Inc.; J.C. is a scientific founder and the chief scientific officer of Genovel Biotech Corp. Both hold equities with their corresponding company.</p><p></p></div></div><div id="ref-list-1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ref-list-1title">References:</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">Egger G, Liang G, Aparicio A &amp; Jones PA
<span class="ref-title">Epigenetics in human disease and prospects for epigenetic therapy</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">429</span>, 457–463 (2004) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15164071" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Epigenetics+in+human+disease+and+prospects+for+epigenetic+therapy&amp;author=G+Egger&amp;author=G+Liang&amp;author=A+Aparicio&amp;author=PA+Jones&amp;volume=429&amp;publication_year=2004&amp;pages=457-463&amp;pmid=15164071&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">Wang X
et al.
<span class="ref-title">N6-methyladenosine-dependent regulation of messenger RNA stability</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">505</span>, 117–120 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3877715/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24284625" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=N6-methyladenosine-dependent+regulation+of+messenger+RNA+stability&amp;author=X+Wang&amp;volume=505&amp;publication_year=2014&amp;pages=117-120&amp;pmid=24284625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">Wang X
et al.
<span class="ref-title">N(6)-methyladenosine Modulates Messenger RNA Translation Efficiency</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">161</span>, 1388–1399 (2015) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4825696/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26046440" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=N(6)-methyladenosine+Modulates+Messenger+RNA+Translation+Efficiency&amp;author=X+Wang&amp;volume=161&amp;publication_year=2015&amp;pages=1388-1399&amp;pmid=26046440&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">Huang H
et al.
<span class="ref-title">Recognition of RNA N(6)-methyladenosine by IGF2BP proteins enhances mRNA stability and translation</span>. <span class="ref-journal">Nat Cell Biol</span>
<span class="ref-vol">20</span>, 285–295 (2018) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5826585/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29476152" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=Recognition+of+RNA+N(6)-methyladenosine+by+IGF2BP+proteins+enhances+mRNA+stability+and+translation&amp;author=H+Huang&amp;volume=20&amp;publication_year=2018&amp;pages=285-295&amp;pmid=29476152&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">Batista PJ
et al.
<span class="ref-title">m(6)A RNA modification controls cell fate transition in mammalian embryonic stem cells</span>. <span class="ref-journal">Cell Stem Cell</span>
<span class="ref-vol">15</span>, 707–719 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4278749/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25456834" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=m(6)A+RNA+modification+controls+cell+fate+transition+in+mammalian+embryonic+stem+cells&amp;author=PJ+Batista&amp;volume=15&amp;publication_year=2014&amp;pages=707-719&amp;pmid=25456834&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">Wang Y
et al.
<span class="ref-title">N6-methyladenosine modification destabilizes developmental regulators in embryonic stem cells</span>. <span class="ref-journal">Nat Cell Biol</span>
<span class="ref-vol">16</span>, 191–198 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4640932/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24394384" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=N6-methyladenosine+modification+destabilizes+developmental+regulators+in+embryonic+stem+cells&amp;author=Y+Wang&amp;volume=16&amp;publication_year=2014&amp;pages=191-198&amp;pmid=24394384&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">Geula S
et al.
<span class="ref-title">m6A mRNA methylation facilitates resolution of naive pluripotency toward differentiation</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">347</span>, 1002–1006 (2015) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25569111" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=m6A+mRNA+methylation+facilitates+resolution+of+naive+pluripotency+toward+differentiation&amp;author=S+Geula&amp;volume=347&amp;publication_year=2015&amp;pages=1002-1006&amp;pmid=25569111&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">Xiang Y
et al.
<span class="ref-title">RNA m(6)A methylation regulates the ultraviolet-induced DNA damage response</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">543</span>, 573–576 (2017) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5490984/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28297716" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=RNA+m(6)A+methylation+regulates+the+ultraviolet-induced+DNA+damage+response&amp;author=Y+Xiang&amp;volume=543&amp;publication_year=2017&amp;pages=573-576&amp;pmid=28297716&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">Li Z
et al.
<span class="ref-title">FTO Plays an Oncogenic Role in Acute Myeloid Leukemia as a N(6)-Methyladenosine RNA Demethylase</span>. <span class="ref-journal">Cancer Cell</span>
<span class="ref-vol">31</span>, 127–141 (2017) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5234852/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28017614" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=FTO+Plays+an+Oncogenic+Role+in+Acute+Myeloid+Leukemia+as+a+N(6)-Methyladenosine+RNA+Demethylase&amp;author=Z+Li&amp;volume=31&amp;publication_year=2017&amp;pages=127-141&amp;pmid=28017614&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">Weng H
et al.
<span class="ref-title">METTL14 Inhibits Hematopoietic Stem/Progenitor Differentiation and Promotes Leukemogenesis via mRNA m(6)A Modification</span>. <span class="ref-journal">Cell stem cell</span>
<span class="ref-vol">22</span>, 191–205 e199 (2018) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5860916/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29290617" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+stem+cell&amp;title=METTL14+Inhibits+Hematopoietic+Stem/Progenitor+Differentiation+and+Promotes+Leukemogenesis+via+mRNA+m(6)A+Modification&amp;author=H+Weng&amp;volume=22&amp;publication_year=2018&amp;pages=191-205+e199&amp;pmid=29290617&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">Su R
et al.
<span class="ref-title">R-2HG Exhibits Anti-tumor Activity by Targeting FTO/m(6)A/MYC/CEBPA Signaling</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">172</span>, 90–105 e123 (2018) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5766423/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29249359" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=R-2HG+Exhibits+Anti-tumor+Activity+by+Targeting+FTO/m(6)A/MYC/CEBPA+Signaling&amp;author=R+Su&amp;volume=172&amp;publication_year=2018&amp;pages=90-105+e123&amp;pmid=29249359&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">Deng X, Su R, Feng X, Wei M &amp; Chen J
<span class="ref-title">Role of N6-methyladenosine modification in cancer</span>. <span class="ref-journal">Curr Opin Genet Dev</span>
<span class="ref-vol">48</span>, 1–7 (2017) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5869081/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29040886" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Role+of+N6-methyladenosine+modification+in+cancer&amp;author=X+Deng&amp;author=R+Su&amp;author=X+Feng&amp;author=M+Wei&amp;author=J+Chen&amp;volume=48&amp;publication_year=2017&amp;pages=1-7&amp;pmid=29040886&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Meyer KD
et al.
<span class="ref-title">Comprehensive analysis of mRNA methylation reveals enrichment in 3’ UTRs and near stop codons</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">149</span>, 1635–1646 (2012) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383396/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22608085" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive+analysis+of+mRNA+methylation+reveals+enrichment+in+3%E2%80%99+UTRs+and+near+stop+codons&amp;author=KD+Meyer&amp;volume=149&amp;publication_year=2012&amp;pages=1635-1646&amp;pmid=22608085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">Dominissini D
et al.
<span class="ref-title">Topology of the human and mouse m6A RNA methylomes revealed by m6A-seq</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">485</span>, 201–206 (2012) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22575960" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Topology+of+the+human+and+mouse+m6A+RNA+methylomes+revealed+by+m6A-seq&amp;author=D+Dominissini&amp;volume=485&amp;publication_year=2012&amp;pages=201-206&amp;pmid=22575960&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">Linder B
et al.
<span class="ref-title">Single-nucleotide-resolution mapping of m6A and m6Am throughout the transcriptome</span>. <span class="ref-journal">Nat Methods</span>
<span class="ref-vol">12</span>, 767–772 (2015) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4487409/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26121403" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Single-nucleotide-resolution+mapping+of+m6A+and+m6Am+throughout+the+transcriptome&amp;author=B+Linder&amp;volume=12&amp;publication_year=2015&amp;pages=767-772&amp;pmid=26121403&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">Liu J
et al.
<span class="ref-title">A METTL3-METTL14 complex mediates mammalian nuclear RNA N6-adenosine methylation</span>. <span class="ref-journal">Nat Chem Biol</span>
<span class="ref-vol">10</span>, 93–95 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911877/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24316715" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Chem+Biol&amp;title=A+METTL3-METTL14+complex+mediates+mammalian+nuclear+RNA+N6-adenosine+methylation&amp;author=J+Liu&amp;volume=10&amp;publication_year=2014&amp;pages=93-95&amp;pmid=24316715&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Barski A
et al.
<span class="ref-title">High-resolution profiling of histone methylations in the human genome</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">129</span>, 823–837 (2007) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17512414" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&amp;author=A+Barski&amp;volume=129&amp;publication_year=2007&amp;pages=823-837&amp;pmid=17512414&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">Pokholok DK
et al.
<span class="ref-title">Genome-wide map of nucleosome acetylation and methylation in yeast</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">122</span>, 517–527 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16122420" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Genome-wide+map+of+nucleosome+acetylation+and+methylation+in+yeast&amp;author=DK+Pokholok&amp;volume=122&amp;publication_year=2005&amp;pages=517-527&amp;pmid=16122420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">Sun XJ
et al.
<span class="ref-title">Identification and characterization of a novel human histone H3 lysine 36-specific methyltransferase</span>. <span class="ref-journal">The Journal of biological chemistry</span>
<span class="ref-vol">280</span>, 35261–35271 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16118227" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+biological+chemistry&amp;title=Identification+and+characterization+of+a+novel+human+histone+H3+lysine+36-specific+methyltransferase&amp;author=XJ+Sun&amp;volume=280&amp;publication_year=2005&amp;pages=35261-35271&amp;pmid=16118227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">Whetstine JR
et al.
<span class="ref-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">125</span>, 467–481 (2006) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16603238" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases&amp;author=JR+Whetstine&amp;volume=125&amp;publication_year=2006&amp;pages=467-481&amp;pmid=16603238&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="mixed-citation">Carrozza MJ
et al.
<span class="ref-title">Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">123</span>, 581–592 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16286007" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Histone+H3+methylation+by+Set2+directs+deacetylation+of+coding+regions+by+Rpd3S+to+suppress+spurious+intragenic+transcription&amp;author=MJ+Carrozza&amp;volume=123&amp;publication_year=2005&amp;pages=581-592&amp;pmid=16286007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="mixed-citation">Joshi AA &amp; Struhl K
<span class="ref-title">Eaf3 chromodomain interaction with methylated H3-K36 links histone deacetylation to Pol II elongation</span>. <span class="ref-journal">Molecular cell</span>
<span class="ref-vol">20</span>, 971–978 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16364921" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+cell&amp;title=Eaf3+chromodomain+interaction+with+methylated+H3-K36+links+histone+deacetylation+to+Pol+II+elongation&amp;author=AA+Joshi&amp;author=K+Struhl&amp;volume=20&amp;publication_year=2005&amp;pages=971-978&amp;pmid=16364921&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="mixed-citation">Luco RF
et al.
<span class="ref-title">Regulation of alternative splicing by histone modifications</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">327</span>, 996–1000 (2010) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2913848/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20133523" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Regulation+of+alternative+splicing+by+histone+modifications&amp;author=RF+Luco&amp;volume=327&amp;publication_year=2010&amp;pages=996-1000&amp;pmid=20133523&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="mixed-citation">Dominguez AA, Lim WA &amp; Qi LS
<span class="ref-title">Beyond editing: repurposing CRISPR-Cas9 for precision genome regulation and interrogation</span>. <span class="ref-journal">Nature reviews. Molecular cell biology</span>
<span class="ref-vol">17</span>, 5–15 (2016) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922510/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26670017" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+reviews.+Molecular+cell+biology&amp;title=Beyond+editing:+repurposing+CRISPR-Cas9+for+precision+genome+regulation+and+interrogation&amp;author=AA+Dominguez&amp;author=WA+Lim&amp;author=LS+Qi&amp;volume=17&amp;publication_year=2016&amp;pages=5-15&amp;pmid=26670017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="mixed-citation">Kizer KO
et al.
<span class="ref-title">A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation</span>. <span class="ref-journal">Mol Cell Biol</span>
<span class="ref-vol">25</span>, 3305–3316 (2005) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1069628/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15798214" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=A+novel+domain+in+Set2+mediates+RNA+polymerase+II+interaction+and+couples+histone+H3+K36+methylation+with+transcript+elongation&amp;author=KO+Kizer&amp;volume=25&amp;publication_year=2005&amp;pages=3305-3316&amp;pmid=15798214&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="mixed-citation">Olmos Y, Hodgson L, Mantell J, Verkade P &amp; Carlton JG
<span class="ref-title">ESCRT-III controls nuclear envelope reformation</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">522</span>, 236–239 (2015) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471131/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26040713" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=ESCRT-III+controls+nuclear+envelope+reformation&amp;author=Y+Olmos&amp;author=L+Hodgson&amp;author=J+Mantell&amp;author=P+Verkade&amp;author=JG+Carlton&amp;volume=522&amp;publication_year=2015&amp;pages=236-239&amp;pmid=26040713&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="mixed-citation">Wienken CJ, Baaske P, Rothbauer U, Braun D &amp; Duhr S
<span class="ref-title">Protein-binding assays in biological liquids using microscale thermophoresis</span>. <span class="ref-journal">Nature communications</span>
<span class="ref-vol">1</span>, 100 (2010) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20981028" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+communications&amp;title=Protein-binding+assays+in+biological+liquids+using+microscale+thermophoresis&amp;author=CJ+Wienken&amp;author=P+Baaske&amp;author=U+Rothbauer&amp;author=D+Braun&amp;author=S+Duhr&amp;volume=1&amp;publication_year=2010&amp;pages=100&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="mixed-citation">Hacker KE
et al.
<span class="ref-title">Structure/Function Analysis of Recurrent Mutations in SETD2 Protein Reveals a Critical and Conserved Role for a SET Domain Residue in Maintaining Protein Stability and Histone H3 Lys-36 Trimethylation</span>. <span class="ref-journal">The Journal of biological chemistry</span>
<span class="ref-vol">291</span>, 21283–21295 (2016) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5076534/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27528607" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+biological+chemistry&amp;title=Structure/Function+Analysis+of+Recurrent+Mutations+in+SETD2+Protein+Reveals+a+Critical+and+Conserved+Role+for+a+SET+Domain+Residue+in+Maintaining+Protein+Stability+and+Histone+H3+Lys-36+Trimethylation&amp;author=KE+Hacker&amp;volume=291&amp;publication_year=2016&amp;pages=21283-21295&amp;pmid=27528607&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 201.992px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#ABS1title">Abstract</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#S1title">Extended Data</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#SM1title">Supplementary Material</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#S2title">Acknowledgements:</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#fn-group-1title">Footnotes</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#ref-list-1title">References:</a></li></ul></div></div><!--post-content--><div id="body-link-poppers"><span></span><div id="body-link-popper-R1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">1. <span class="mixed-citation">Egger G, Liang G, Aparicio A &amp; Jones PA
<span class="ref-title">Epigenetics in human disease and prospects for epigenetic therapy</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">429</span>, 457–463 (2004) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15164071" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Epigenetics+in+human+disease+and+prospects+for+epigenetic+therapy&amp;author=G+Egger&amp;author=G+Liang&amp;author=A+Aparicio&amp;author=PA+Jones&amp;volume=429&amp;publication_year=2004&amp;pages=457-463&amp;pmid=15164071&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R1">Ref list</a>]</div><div id="body-link-popper-R2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">2. <span class="mixed-citation">Wang X
et al.
<span class="ref-title">N6-methyladenosine-dependent regulation of messenger RNA stability</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">505</span>, 117–120 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3877715/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24284625" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=N6-methyladenosine-dependent+regulation+of+messenger+RNA+stability&amp;author=X+Wang&amp;volume=505&amp;publication_year=2014&amp;pages=117-120&amp;pmid=24284625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R2">Ref list</a>]</div><div id="body-link-popper-R4" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">4. <span class="mixed-citation">Huang H
et al.
<span class="ref-title">Recognition of RNA N(6)-methyladenosine by IGF2BP proteins enhances mRNA stability and translation</span>. <span class="ref-journal">Nat Cell Biol</span>
<span class="ref-vol">20</span>, 285–295 (2018) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5826585/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29476152" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=Recognition+of+RNA+N(6)-methyladenosine+by+IGF2BP+proteins+enhances+mRNA+stability+and+translation&amp;author=H+Huang&amp;volume=20&amp;publication_year=2018&amp;pages=285-295&amp;pmid=29476152&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R4">Ref list</a>]</div><div id="body-link-popper-R5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">5. <span class="mixed-citation">Batista PJ
et al.
<span class="ref-title">m(6)A RNA modification controls cell fate transition in mammalian embryonic stem cells</span>. <span class="ref-journal">Cell Stem Cell</span>
<span class="ref-vol">15</span>, 707–719 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4278749/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25456834" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=m(6)A+RNA+modification+controls+cell+fate+transition+in+mammalian+embryonic+stem+cells&amp;author=PJ+Batista&amp;volume=15&amp;publication_year=2014&amp;pages=707-719&amp;pmid=25456834&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R5">Ref list</a>]</div><div id="body-link-popper-R12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">12. <span class="mixed-citation">Deng X, Su R, Feng X, Wei M &amp; Chen J
<span class="ref-title">Role of N6-methyladenosine modification in cancer</span>. <span class="ref-journal">Curr Opin Genet Dev</span>
<span class="ref-vol">48</span>, 1–7 (2017) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5869081/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29040886" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Role+of+N6-methyladenosine+modification+in+cancer&amp;author=X+Deng&amp;author=R+Su&amp;author=X+Feng&amp;author=M+Wei&amp;author=J+Chen&amp;volume=48&amp;publication_year=2017&amp;pages=1-7&amp;pmid=29040886&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R12">Ref list</a>]</div><div id="body-link-popper-R13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">13. <span class="mixed-citation">Meyer KD
et al.
<span class="ref-title">Comprehensive analysis of mRNA methylation reveals enrichment in 3’ UTRs and near stop codons</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">149</span>, 1635–1646 (2012) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383396/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22608085" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive+analysis+of+mRNA+methylation+reveals+enrichment+in+3%E2%80%99+UTRs+and+near+stop+codons&amp;author=KD+Meyer&amp;volume=149&amp;publication_year=2012&amp;pages=1635-1646&amp;pmid=22608085&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R13">Ref list</a>]</div><div id="body-link-popper-R15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">15. <span class="mixed-citation">Linder B
et al.
<span class="ref-title">Single-nucleotide-resolution mapping of m6A and m6Am throughout the transcriptome</span>. <span class="ref-journal">Nat Methods</span>
<span class="ref-vol">12</span>, 767–772 (2015) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4487409/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26121403" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Single-nucleotide-resolution+mapping+of+m6A+and+m6Am+throughout+the+transcriptome&amp;author=B+Linder&amp;volume=12&amp;publication_year=2015&amp;pages=767-772&amp;pmid=26121403&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R15">Ref list</a>]</div><div id="body-link-popper-R16" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">16. <span class="mixed-citation">Liu J
et al.
<span class="ref-title">A METTL3-METTL14 complex mediates mammalian nuclear RNA N6-adenosine methylation</span>. <span class="ref-journal">Nat Chem Biol</span>
<span class="ref-vol">10</span>, 93–95 (2014) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911877/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24316715" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Chem+Biol&amp;title=A+METTL3-METTL14+complex+mediates+mammalian+nuclear+RNA+N6-adenosine+methylation&amp;author=J+Liu&amp;volume=10&amp;publication_year=2014&amp;pages=93-95&amp;pmid=24316715&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R16">Ref list</a>]</div><div id="body-link-popper-R17" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">17. <span class="mixed-citation">Barski A
et al.
<span class="ref-title">High-resolution profiling of histone methylations in the human genome</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">129</span>, 823–837 (2007) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17512414" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=High-resolution+profiling+of+histone+methylations+in+the+human+genome&amp;author=A+Barski&amp;volume=129&amp;publication_year=2007&amp;pages=823-837&amp;pmid=17512414&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R17">Ref list</a>]</div><div id="body-link-popper-R18" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">18. <span class="mixed-citation">Pokholok DK
et al.
<span class="ref-title">Genome-wide map of nucleosome acetylation and methylation in yeast</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">122</span>, 517–527 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16122420" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Genome-wide+map+of+nucleosome+acetylation+and+methylation+in+yeast&amp;author=DK+Pokholok&amp;volume=122&amp;publication_year=2005&amp;pages=517-527&amp;pmid=16122420&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R18">Ref list</a>]</div><div id="body-link-popper-R19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">19. <span class="mixed-citation">Sun XJ
et al.
<span class="ref-title">Identification and characterization of a novel human histone H3 lysine 36-specific methyltransferase</span>. <span class="ref-journal">The Journal of biological chemistry</span>
<span class="ref-vol">280</span>, 35261–35271 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16118227" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+biological+chemistry&amp;title=Identification+and+characterization+of+a+novel+human+histone+H3+lysine+36-specific+methyltransferase&amp;author=XJ+Sun&amp;volume=280&amp;publication_year=2005&amp;pages=35261-35271&amp;pmid=16118227&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R19">Ref list</a>]</div><div id="body-link-popper-R20" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">20. <span class="mixed-citation">Whetstine JR
et al.
<span class="ref-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">125</span>, 467–481 (2006) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16603238" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases&amp;author=JR+Whetstine&amp;volume=125&amp;publication_year=2006&amp;pages=467-481&amp;pmid=16603238&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R20">Ref list</a>]</div><div id="body-link-popper-R28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">28. <span class="mixed-citation">Hacker KE
et al.
<span class="ref-title">Structure/Function Analysis of Recurrent Mutations in SETD2 Protein Reveals a Critical and Conserved Role for a SET Domain Residue in Maintaining Protein Stability and Histone H3 Lys-36 Trimethylation</span>. <span class="ref-journal">The Journal of biological chemistry</span>
<span class="ref-vol">291</span>, 21283–21295 (2016) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5076534/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27528607" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+biological+chemistry&amp;title=Structure/Function+Analysis+of+Recurrent+Mutations+in+SETD2+Protein+Reveals+a+Critical+and+Conserved+Role+for+a+SET+Domain+Residue+in+Maintaining+Protein+Stability+and+Histone+H3+Lys-36+Trimethylation&amp;author=KE+Hacker&amp;volume=291&amp;publication_year=2016&amp;pages=21283-21295&amp;pmid=27528607&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R28">Ref list</a>]</div><div id="body-link-popper-R25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">25. <span class="mixed-citation">Kizer KO
et al.
<span class="ref-title">A novel domain in Set2 mediates RNA polymerase II interaction and couples histone H3 K36 methylation with transcript elongation</span>. <span class="ref-journal">Mol Cell Biol</span>
<span class="ref-vol">25</span>, 3305–3316 (2005) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1069628/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15798214" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=A+novel+domain+in+Set2+mediates+RNA+polymerase+II+interaction+and+couples+histone+H3+K36+methylation+with+transcript+elongation&amp;author=KO+Kizer&amp;volume=25&amp;publication_year=2005&amp;pages=3305-3316&amp;pmid=15798214&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R25">Ref list</a>]</div><div id="body-link-popper-R21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">21. <span class="mixed-citation">Carrozza MJ
et al.
<span class="ref-title">Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">123</span>, 581–592 (2005) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16286007" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Histone+H3+methylation+by+Set2+directs+deacetylation+of+coding+regions+by+Rpd3S+to+suppress+spurious+intragenic+transcription&amp;author=MJ+Carrozza&amp;volume=123&amp;publication_year=2005&amp;pages=581-592&amp;pmid=16286007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R21">Ref list</a>]</div><div id="body-link-popper-R23" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">23. <span class="mixed-citation">Luco RF
et al.
<span class="ref-title">Regulation of alternative splicing by histone modifications</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">327</span>, 996–1000 (2010) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2913848/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20133523" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Regulation+of+alternative+splicing+by+histone+modifications&amp;author=RF+Luco&amp;volume=327&amp;publication_year=2010&amp;pages=996-1000&amp;pmid=20133523&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R23">Ref list</a>]</div><div id="body-link-popper-R24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">24. <span class="mixed-citation">Dominguez AA, Lim WA &amp; Qi LS
<span class="ref-title">Beyond editing: repurposing CRISPR-Cas9 for precision genome regulation and interrogation</span>. <span class="ref-journal">Nature reviews. Molecular cell biology</span>
<span class="ref-vol">17</span>, 5–15 (2016) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922510/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26670017" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+reviews.+Molecular+cell+biology&amp;title=Beyond+editing:+repurposing+CRISPR-Cas9+for+precision+genome+regulation+and+interrogation&amp;author=AA+Dominguez&amp;author=WA+Lim&amp;author=LS+Qi&amp;volume=17&amp;publication_year=2016&amp;pages=5-15&amp;pmid=26670017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R24">Ref list</a>]</div><div id="body-link-popper-R26" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">26. <span class="mixed-citation">Olmos Y, Hodgson L, Mantell J, Verkade P &amp; Carlton JG
<span class="ref-title">ESCRT-III controls nuclear envelope reformation</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">522</span>, 236–239 (2015) <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471131/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26040713" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=ESCRT-III+controls+nuclear+envelope+reformation&amp;author=Y+Olmos&amp;author=L+Hodgson&amp;author=J+Mantell&amp;author=P+Verkade&amp;author=JG+Carlton&amp;volume=522&amp;publication_year=2015&amp;pages=236-239&amp;pmid=26040713&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R26">Ref list</a>]</div><div id="body-link-popper-R27" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">27. <span class="mixed-citation">Wienken CJ, Baaske P, Rothbauer U, Braun D &amp; Duhr S
<span class="ref-title">Protein-binding assays in biological liquids using microscale thermophoresis</span>. <span class="ref-journal">Nature communications</span>
<span class="ref-vol">1</span>, 100 (2010) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20981028" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+communications&amp;title=Protein-binding+assays+in+biological+liquids+using+microscale+thermophoresis&amp;author=CJ+Wienken&amp;author=P+Baaske&amp;author=U+Rothbauer&amp;author=D+Braun&amp;author=S+Duhr&amp;volume=1&amp;publication_year=2010&amp;pages=100&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R27">Ref list</a>]</div><div id="body-link-popper-R7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 446px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">7. <span class="mixed-citation">Geula S
et al.
<span class="ref-title">m6A mRNA methylation facilitates resolution of naive pluripotency toward differentiation</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">347</span>, 1002–1006 (2015) [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25569111" ref="reftype=pubmed&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=m6A+mRNA+methylation+facilitates+resolution+of+naive+pluripotency+toward+differentiation&amp;author=S+Geula&amp;volume=347&amp;publication_year=2015&amp;pages=1002-1006&amp;pmid=25569111&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6438714&amp;issue-id=331668&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#R7">Ref list</a>]</div></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  


    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/pdf/nihms-1521376.pdf" class="int-view">PDF (6.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li class="citation-link other_item"><a class="citationexporter ctxp" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic#" data-citationid="PMC6438714" role="button" aria-expanded="false" aria-haspopup="true">Cite</a></li>

            <li>
                
                    




    <div class="favorites-button-container" id="pmc-favorites-button" data-toggle-favorites-enabled="false" data-add-to-favorites-url-hash="#add-to-favorites" data-in-favorites="false" data-toggle-url="https://www.ncbi.nlm.nih.gov/pmc/toggle-favorites-collection/PMC6438714/" data-collections-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/" data-remove-from-favorites-text="Remove from Favorites collection" data-add-to-favorites-text="Add to Favorites collection" data-display-title-attr="Favorites">
        
            <a href="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A//www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/%3Freport%3Dclassic%23add-to-favorites" class="usa-button favorites-button favorites-button-empty favorites-button-link" id="pmc-favorites-button-link" aria-label="Add to Favorites collection">
                Favorites
            </a>
        
    </div>


                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F&amp;text=Histone%20H3%20trimethylation%20at%20lysine%2036%20guides%20m6A%20RNA%20modification%20co-transcriptionally" alt="Share on Twitter" aria-expanded="false" aria-haspopup="true"><i class="fa fa-twitter fa-stack-1x">&nbsp;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F" alt="Share on Facebook" aria-expanded="false" aria-haspopup="true"><i class="fa fa-facebook fa-stack-1x">&nbsp;</i></a></li>
                <li>
                    <div class="share-permalink dropdown-block">
                        <button class="trigger" alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg">
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" aria-hidden="true">
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6438714/?report=classic" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    
        <section>
            <h6>RESOURCES</h6>
            <ul class="pmc-sidebar__pubmed_links">
                <li><a href="https://pubmed.ncbi.nlm.nih.gov/?linkname=pubmed_pubmed&amp;from_uid=30867593">Similar articles in PubMed</a></li>
            </ul>
        </section>
    


    </aside>
</main>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11" d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11" d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin" href="https://www.linkedin.com/company/ncbinlm" aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11" d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11" d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12" d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12" d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12" d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12" d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12" d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12" d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12" d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"></path></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter" target="_blank" rel="noopener noreferrer" role="button" aria-expanded="false" aria-haspopup="true"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine" aria-label="Facebook" rel="noopener noreferrer" target="_blank" role="button" aria-expanded="false" aria-haspopup="true">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube" target="_blank" rel="noopener noreferrer" role="button" aria-expanded="false" aria-haspopup="true"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5" d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323" class="text-white" target="_blank" rel="noopener noreferrer" role="button" aria-expanded="false" aria-haspopup="true">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="https://www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="./pubmed_files/output.0f72d6a64937.js"></script>
  
  
    <script src="./pubmed_files/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="./pubmed_files/output.a212a9fcf845.js"></script>
<script src="./pubmed_files/output.7999321d1aac.js"></script>
<script src="./pubmed_files/output.7ca436b2ea51.js"></script>
<script src="./pubmed_files/output.f8422046fbe0.js"></script>
<script src="./pubmed_files/output.ff40c7d85ff8.js"></script>
<script src="./pubmed_files/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="./pubmed_files/base.f0b3c28f5fddaecf2c49.js"></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="./pubmed_files/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="./pubmed_files/jig.nojquery.min.js">//</script><script type="text/javascript" src="./pubmed_files/common.min.js">//</script><script type="text/javascript" src="./pubmed_files/NcbiTagServer.min.js">//</script><script type="text/javascript" src="./pubmed_files/crb.min.js">//</script><script type="text/javascript" src="./pubmed_files/jactions.min.js">//</script><meta name="citationexporter" content="backend:&#39;https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/&#39;"><script type="text/javascript" src="./pubmed_files/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="./pubmed_files/citationexporter.css" type="text/css"><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script type="text/javascript" src="./pubmed_files/favorite-button.js"></script>
    <script type="text/javascript" src="./pubmed_files/article.1e76407cf1fa37e81de0.js"></script>
    <script type="text/javascript">
        var FavoritesButton = exports.FavoritesButton;
        var favButton = new FavoritesButton($('.favorites-button-container'));
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC6438714/'});
    </script>


  
  
  <script type="text/javascript" src="./pubmed_files/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


  
        
        <a class="custom-feedback-button custom-feedback-button--base" href="https://support.nlm.nih.gov/support/create-case/?from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6438714%2F%3Freport%3Dclassic" ref="linksrc=feedback_btn" rel="noopener noreferrer nofollow" target="_blank">Feedback</a>
  


<div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><script type="text/javascript" src="./pubmed_files/saved_resource"></script></body></html>